In [1]:
from vampyr import vampyr3d as vp
import numpy as np
import matplotlib.pyplot as plt
# from copy import deepcopy

# import KAIN
import utils
# from spinor import spinor
from scfsolv import scfsolv as ss
from scfsolv import scfsolv_1stpert as ss1

prec = 1.0e-4
khist = 4
thrs = prec
c =  137.035999177
# c = 60

N_components = 2
# Pord = 1 #maximum pertubation order computed

# r_x = np.linspace(-16, 16, 10000) # create an evenly spaced set of points between -0.99 and 0.99
# r_y = np.linspace(-16, 16, 10000)
# r_z = np.linspace(-16, 16, 10000)
# r = [r_x, r_y, r_z]
# Restricted = True


# Electric_field = [0.01, 0., 0.]
Electric_field = [0.01, 0., 0.]
# Analytic nuclear potential
# print("pouet")
#H
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[0.1, 0.100, 0.100]], [1], "/home/qpitto/Tests_KAIN/test_H/norun/orbitals/", 1, 1, False #Initial guess 
#He 
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[0.1, 0.100, 0.100]], [2], "/home/qpitto/Tests_KAIN/test_He/norun/orbitals/", 2, 1, False #Initial guess
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[0.1, 0.100, 0.100]], [2], "/home/qpitto/Tests_KAIN/test_He/run/orbitals/", 2, 2, False #Converged NR result
#H2
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[0.73699313, 0.100, 0.100], [-0.73699313, 0.100, 0.100]], [1,1], "/home/qpitto/Tests_KAIN/test_H2/norun/orbitals/", 2, 1, False #Initial guess
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[1., 0.100, 0.100], [-1., 0.100, 0.100]], [1,1], "/home/qpitto/Tests_KAIN/test_H2/norun/orbitals/", 2, 1, False #Initial guess
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[0.7, 0.100, 0.100], [-0.7, 0.100, 0.100]], [1,1], "/home/qpitto/Tests_KAIN/test_H2/norun/orbitals/", 2, 1, False #Initial guess
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[0.6992, 0.100, 0.100], [-0.6992, 0.100, 0.100]], [1,1], "/home/qpitto/Tests_KAIN/test_H2/norun/orbitals/", 2, 1, False #Initial guess
#H2+ 
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[1., 0.100, 0.100], [-1., 0.100, 0.100]], [1,1], "/home/qpitto/Tests_KAIN/test_H2/norun/orbitals/", 1, 1, False #Initial guess
# H2plus_dirac = -1.1026415810325771641
# lambda_factor = (2*c**2 + H2plus_dirac)/(2*c**2)
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[1/lambda_factor, 0.100, 0.100], [-1/lambda_factor, 0.100, 0.100]], [1,1], "/home/qpitto/Tests_KAIN/test_H2/norun/orbitals/", 1, 1, False #Initial guess
#LiH
# R, Z, pathMol, nElectron = [[1.508001,  0.188973, 0.188973], [-1.508001,  0.188973, 0.188973]], [3,1], "/home/qpitto/Tests_KAIN/test_LiH/norun/orbitals/", 2 #Initial guess
# R, Z, pathMol, nElectron = [[1.508001,  0.188973, 0.188973], [-1.508001,  0.188973, 0.188973]], [3,1], "/home/qpitto/Tests_KAIN/test_LiH/run/orbitals/", 2 #LiH
R, Z, pathMol, nElectron, InitGuessType, scalar = [[1.508001,  0.188973, 0.188973], [-1.508001,  0.188973, 0.188973]], [3,1], "/home/qpitto/Tests_KAIN/test_LiH/run/orbitals/", 4, 2, False
#Ne
# R, Z, pathMol, nElectron = [[0.100, 0.100, 0.100]], [10], "/home/qpitto/Tests_KAIN/test_Ne/norun/orbitals/phi_p_scf_", 5 #Ne
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[0.1, 0.100, 0.100]], [10], "/home/qpitto/Tests_KAIN/test_He/norun/orbitals/", 2, 1, False #Ne8+
#C
# R, Z, pathMol, nElectron, InitGuessType, scalar = [[0.1, 0.100, 0.100]], [6], "/home/qpitto/Tests_KAIN/test_C/norun/orbitals/", 6, 2, False
#H2O
# R, Z, pathMol, nElectron = [[0.0000,  0.000, -0.125], [-1.4375,  0.000,  1.025], [1.4375,  0.000,  1.025]], [8,1,1], "/home/qpitto/Tests_KAIN/test_water/run/orbitals/", 5 #H2O
# R, Z, pathMol, nElectron = [[0.0000,  0.000, -0.125], [-1.4375,  0.000,  1.025], [1.4375,  0.000,  1.025]], [8,1,1], "/home/qpitto/Tests_KAIN/initial_guesses/phi_p_scf_", 5 #H2O
#CH4
# R, Z, pathMol, nElectron = [[0.1, 0.1, 0.1], [0.1, 0.1, 2.15413215], [2.03631019, 0.1, -0.5856834], [-0.54635362, 1.92524633, -0.5856834], [-0.54635362, -1.72524633, -0.5856834]], [6,1,1,1,1], "/home/qpitto/Tests_KAIN/test_CH4/norun/orbitals/", 5 #CH4

nz = len(Z)

#create scf solver instance with default MRA
Scfs = ss(N_components, prec, khist, scalar=scalar, lightspeed=c)

# print("testLC: ", utils.LeviCivita([1,2,3]), utils.LeviCivita([1,1,3]), utils.LeviCivita([2,1,3]))
# from spinor import spinor
# print("Test spinors")
# spineur = spinor(Scfs.mra, 2)
# spineur.setZero()
# print("length", len(spineur))
# spindeur = spineur + spineur
# print("+ ok")
# spindeur = spineur - spineur
# print("- ok")
# spindeur = 1. * spineur
# spindeur = 2 * spineur
# spindeur = 1j * spineur
# spindeur = spineur * 1
# print("* ok")
# spindeur = spineur / 2
# spindeur = spineur / 2.
# print("norme = ", spineur.compSqNorm())
# print("dot = ", spineur.dot(spindeur))
# spindeur = spineur.derivative(0)
# print("Derivative ok")


# tut = [[-1.514986e+00, -8.879809e-02],
# [-8.879809e-02, 1.188349e+00]]
# print(np.linalg.eig(tut))

#initialise molecule geometry and properties
print("start initMolec")
Scfs.initMolec(nElectron, R, Z, pathMol, InitGuessType, scalar)
print("init done")
#run the SCF solver
Scfs.scfRun(thrs, True, True)

# Scfs_2C = ss(N_components, 0.01*prec, khist, scalar = False)
# reprojected_copy = Scfs.phi_prev[0][-1].reproject(Scfs_2C.P_eps)
# # print("tutbim")
# # Scfs_2C.initMolec(nElectron, R, Z, Scfs, restricted=False, copy_orbitals=True)
# # Scfs_2C.scfRun(thrs, True, True)

# print("Sanity test")
# for i in range(len(Scfs.phi_prev)):
#     for j in range(len(Scfs.phi_prev)):
#         print(f"Test: {i}, {j}", vp.dot(Scfs.phi_prev[i][-1], Scfs.phi_prev[j][-1]))

# print("Start perturbation")
# Scfs_pert = ss1(Scfs)

# print("Contructor passed")
# Scfs_pert.initMolec(Electric_field)
# # Scfs_pert.print_operators()

# Scfs_pert.scfRun(thrs, True, True)

start initMolec
Fock [[-4.09328238-1.30417070e-07j -0.08921977-5.35490255e-10j]
 [-0.08925085-1.50502278e-08j -0.64610978-7.88844913e-10j]]
Kinetic [[3.60407744-1.30417070e-07j 0.49235698-1.50502278e-08j]
 [0.49236022-5.35490255e-10j 0.39084825-7.88844913e-10j]]
Vnuc [[-8.3770787 +0.j -0.5745742 +0.j]
 [-0.5745742 +0.j -1.85352593+0.j]]
Coulomb [[2.33616525+0.j 0.07511482+0.j]
 [0.07511482+0.j 1.30685301+0.j]]
Exchange [[1.65644636+0.j 0.08214845+0.j]
 [0.08212062+0.j 0.49028512+0.j]]


/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:170: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu.append(vp.HelmholtzOperator(self.mra, mu, self.prec))  # Initalize the operator
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:665: ComplexWarning: Casting complex values to real discards the imaginary part
  VzFphi = ((( V_z/(self.c**2) * self.Fock[orb, orb]).crop(self.prec)) * self.phi_prev[orb][-1] ).crop(self.prec)


====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in (0.9999999999999992+0j) (-0.03139875502321517-8.272450190671197e-09j) (-1.5926061706648322e-09-1.9309688153220865e-15j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.complex128'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe24ac93af0>
Orthonormalise GS (2.1574266240358728e-13-1.2941421328884719e-08j) (7.142704589395394e-22-1.8964952330077275e-15j) (-7.142704589395394e-22+1.8964952330077275e-15j)
init done
scf run
=============Iteration: 0
F= [[-4.09328238-1.30417070e-07j -0.08921977-5.35490255e-10j]
 [-0.08925085-1.50502278e-08j -0.64610978-7.88844913e-10j]]
CompOverlap 4
S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+9.14748818e-16j
   2.15742662e-13-1.29414213e-08j -6.31552398e-22-7.06396609e-17j]
 [ 0.00000000e+00-9.14748818e-16j  1.00000000e+00+0.00000000e+00j
   7.142

/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:498: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu[orb] = vp.HelmholtzOperator(self.mra, mu, self.prec)


CompOverlap 4
ExpandSol pre orthonorm S= [[ 3.89914710e+00+0.00000000e+00j  0.00000000e+00-6.85529878e-15j
  -8.41699026e-02+5.78701418e-08j -4.71134698e-09+2.99023826e-15j]
 [ 0.00000000e+00+6.85529878e-15j  3.89914710e+00+0.00000000e+00j
   4.71134698e-09-1.78469074e-14j -8.41699026e-02-5.78701418e-08j]
 [-8.41699026e-02-5.78701418e-08j  4.71134698e-09+1.78469074e-14j
   3.95164119e+00+0.00000000e+00j  0.00000000e+00-3.55992501e-15j]
 [-4.71134698e-09-2.99023826e-15j -8.41699026e-02+5.78701418e-08j
   0.00000000e+00+3.55992501e-15j  3.95164119e+00+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in (1.9746258136516213+0j) (-0.08416990257063328-5.78701418272684e-08j) (4.711346981489939e-09+1.7846907417576767e-14j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.complex128'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at

KeyboardInterrupt: 

In [ ]:
# Scfs_2C = ss(N_components, 0.01*prec, khist, scalar = False)
# # reprojected_copy = Scfs.phi_prev[0][-1].reproject(Scfs_2C.P_eps)
# print("tutbim")
# Scfs_2C.initMolec(nElectron, R, Z, Scfs, restricted=False, copy_orbitals=True)
# print("===============Init done=============================================")
# Scfs_2C.scfRun(thrs, True, True)

In [ ]:
import numpy as np
# F= [[-2.7896489 -2.66352263e-07j, 0.00707083+5.25880281e-08j],[ 0.00704288-1.01844084e-07j, -1.11828297+1.44609334e-08j]] #initial
F=np.array([[-2.7263095 -9.37847108e-05j, -0.04998474+1.23454107e-03j],
 [-0.05001107-9.24036599e-04j, -1.19366291-3.31073015e-06j]]) #final (no kain)
eigvals, U = np.linalg.eig(F)
print(eigvals)

[-2.72793954-8.35624316e-05j -1.19203287-1.35330094e-05j]


In [ ]:
# # print("Scaled energy (4C Dirac energy)", (1/(-0.50056707) - 1/(c**2))**(-1))
# # mp = 1.672621925e-27 # in kg
# # me = 9.1093837139e-31 #in kg, but we don't need it 
# me = 1 #in A.U. by definition
# mp = 1.672621925e-27/9.1093837139e-31 * me 
# mu = me*mp/(me+mp)  #TODO convertir en unités atomiques
# print(mu)
# alpha = 1/137.035999177
# n,k = 1,-1 #quantum numbers
# c_original = 1/alpha
# gse_H = mu*c_original**2 / np.sqrt(1 + (Scfs.Z[0]*alpha/(n-np.abs(k) + np.sqrt(k**2 - (Scfs.Z[0]*alpha)**2)))**2)
# # print("Dirac ground state energy of Hydrogen",-13.598433/27.211386245988) #On numerator: Ionisation energy of H in eV (source: NIST), denominator: conversion factor
# print("Dirac ground state energy of Hydrogen",gse_H - c_original**2) #On numerator: Ionisation energy of H in eV (source: Wikipedia, Bohr-Sommerfeld model), denominator: conversion factor
print("Ground state of He in atomic units:", -79.02/27.211386245988)
print(2*(1.43079923) + 2*(-3.3747113) + 2.05163705 - 1.02581853)
print(-1.36275874*2)

Ground state of He in atomic units: -2.9039314383202575
-2.8620056199999997
-2.72551748


In [ ]:
#From Luca's orbital4c class
def analytic_1s(light_speed, n, k, Z):
    alpha = 1/light_speed
    gamma = compute_gamma(k,Z,alpha)
    tmp1 = n - np.abs(k) + gamma
    tmp2 = Z * alpha / tmp1
    tmp3 = 1 + tmp2**2
    return light_speed**2 / np.sqrt(tmp3)

def compute_gamma(k,Z,alpha):
    return np.sqrt(k**2 - Z**2 * alpha**2)

c_in = c
n,k = 1,-1 #quantum numbers
# c_in = c_original
gse_H_alt = analytic_1s(c_in, n, k, Scfs.Z[0])
# gse_H_alt = analytic_1s(c_in, n, k, 1)
print(gse_H_alt - c_in**2)

-2.0001065140531864


In [ ]:
Energy_test = -0.50251687
# print((1/Energy_test - 1/(2*137.035999177**2))**(-1))
print((1/Energy_test - 1/(2*10**2))**(-1))

-0.5012574184550684


He 1e-4 c=c

start initMolec
Initmolec
S initial guess =  [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4306249392610726+0j)
Expectation value gradk*Nab :  (-0.00012044563871625121+0j)
Expectation value sporb :  1.0530813269418059e-20j
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4306249392610726+0j)
Expectation value gradk*Nab :  (-0.00012044563871625121+0j)
Expectation value sporb :  1.0530813269418059e-20j
Fock [[-0.75587279-9.16960004e-21j]]
Kinetic [[1.43039976-7.60688169e-21j]]
Vnuc [[-3.21193967+0.j]]
Coulomb [[2.05133425+0.j]]
Exchange [[1.02566713+0.j]]
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:170: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu.append(vp.HelmholtzOperator(self.mra, mu, self.prec))  # Initalize the operator
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:650: ComplexWarning: Casting complex values to real discards the imaginary part
  VzFphi = V_z/(self.c**2) * self.Fock[orb, orb] * self.phi_prev[orb][-1]
Power Iter, norm of new guess =  (1.1636000221646157+0j) (1.1636000220968359+0j) (6.777983127744903e-11+0j)
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
init done
scf run
=============Iteration: 0
F= [[-0.75587279-9.16960004e-21j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.234302644185566+1.2164421130315898e-20j)
Expectation value gradk*Nab :  (-0.0001896295500561754+9.27829789425612e-23j)
Expectation value sporb :  (-1.2258840936673134e-09-7.380082494002417e-10j)
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:505: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu[orb] = vp.HelmholtzOperator(self.mra, mu, self.prec)
Power Iter, norm of new guess =  (0.9545144369677607+0j) (0.9545144369532508+0j) (1.450994868789787e-11+0j)
orthonorm
S durant Ortho [[0.95451444+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.3737119187150915-2.8933942428536424e-14j)
Expectation value gradk*Nab :  (-0.00024151256882254277-4.445931270218589e-14j)
Expectation value sporb :  (2.560565634338832e-10-1.4171918279845763e-10j)
Fock [[-0.92590432+1.18484902e-09j]]
Kinetic [[1.37335916+1.41568302e-10j]]
Vnuc [[-3.30746259-3.63547091e-22j]]
Coulomb [[2.01639822-3.38319294e-22j]]
Exchange [[1.00819911-1.04328071e-09j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.3737119187150915+2.8933942428536424e-14j)
Expectation value gradk*Nab :  (-0.00024151256882254277+4.445931270218589e-14j)
Expectation value sporb :  (3.2221020384703766e-10-1.26383911843977e-10j)
Fock [[-0.92590432-9.16745921e-10j]]
Kinetic [[1.37335916+1.26534792e-10j]]
Vnuc [[-3.30746259+3.63547091e-22j]]
Coulomb [[2.01639822+3.38319294e-22j]]
Exchange [[1.00819911+1.04328071e-09j]]
Orbital: 0    Norm: [0.98572993]    Update: [0.06118879]    Energy:[-0.93510056+7.01888952e-10j]
Scaled energy (4C Dirac energy) [-0.93507728]
=============Iteration: 1
F= [[-0.92590432-9.16745921e-10j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.37361973028909+4.37246138300183e-14j)
Expectation value gradk*Nab :  (-0.00024018197639244103-4.6028255693370674e-14j)
Expectation value sporb :  (2.612019810347377e-10-1.402050945081224e-10j)
Power Iter, norm of new guess =  (0.9839430195371669+0j) (0.983943019534341+0j) (2.825813499590086e-12+0j)
orthonorm
S durant Ortho [[0.98394302+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4277049820484684-2.8428821262221954e-12j)
Expectation value gradk*Nab :  (-0.00025997381502446994+4.010507634517716e-13j)
Expectation value sporb :  (-9.690067467573169e-12+3.3065277030163395e-12j)
Fock [[-0.91819241-5.15848778e-10j]]
Kinetic [[1.4273248-5.7918819e-12j]]
Vnuc [[-3.37092391-5.68685157e-23j]]
Coulomb [[2.05081339-7.61007173e-23j]]
Exchange [[1.02540669+5.10056896e-10j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4277049820484684+2.8428821262221954e-12j)
Expectation value gradk*Nab :  (-0.00025997381502446994-4.010507634517716e-13j)
Expectation value sporb :  (-5.333311036225847e-12+4.279972929752586e-12j)
Fock [[-0.91819241+5.08262278e-10j]]
Kinetic [[1.4273248-1.79461874e-12j]]
Vnuc [[-3.37092391+5.68685157e-23j]]
Coulomb [[2.05081339+7.61007173e-23j]]
Exchange [[1.02540669-5.10056896e-10j]]
Orbital: 0    Norm: [1.00145833]    Update: [0.01703904]    Energy:[-0.92599425+1.183405e-09j]
Scaled energy (4C Dirac energy) [-0.92597142]
=============Iteration: 2
F= [[-0.91819241+5.08262278e-10j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4276055982740383-2.9197647251260125e-12j)
Expectation value gradk*Nab :  (-0.0002584885112870195+4.0382725893439293e-13j)
Expectation value sporb :  (-9.721229913485612e-12+3.2568845232407156e-12j)
Power Iter, norm of new guess =  (0.9989037363648869+0j) (0.998903736364657+0j) (2.2998198792005203e-13+0j)
orthonorm
S durant Ortho [[0.99890374+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4303258964890773+2.8154945952796453e-13j)
Expectation value gradk*Nab :  (-0.00026173652491025983-1.414254251858719e-13j)
Expectation value sporb :  (1.0264391590037613e-11-6.424413012444956e-12j)
Fock [[-0.91815843+6.88959159e-10j]]
Kinetic [[1.42994295+6.56569162e-12j]]
Vnuc [[-3.37359365+7.65142698e-24j]]
Coulomb [[2.05098455-1.15805199e-23j]]
Exchange [[1.02549228-6.82393467e-10j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4303258964890773-2.8154945952796453e-13j)
Expectation value gradk*Nab :  (-0.00026173652491025983+1.414254251858719e-13j)
Expectation value sporb :  (1.3629436514654248e-11-5.539216301476217e-12j)
Fock [[-0.91815843-6.76995529e-10j]]
Kinetic [[1.42994295+5.3979377e-12j]]
Vnuc [[-3.37359365-7.65142698e-24j]]
Coulomb [[2.05098455+1.15805199e-23j]]
Exchange [[1.02549228+6.82393467e-10j]]
Orbital: 0    Norm: [0.99947064]    Update: [0.00121474]    Energy:[-0.91828927-5.15871215e-10j]
Scaled energy (4C Dirac energy) [-0.91826682]
=============Iteration: 3
F= [[-0.91815843-6.76995529e-10j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4302256079736853+3.0267657892122255e-13j)
Expectation value gradk*Nab :  (-0.0002602359558034085-1.4238810454377692e-13j)
Expectation value sporb :  (1.0256621032247844e-11-6.4393809861854615e-12j)
Power Iter, norm of new guess =  (0.9995344453686966+0j) (0.9995344453686966+0j) 0j
orthonorm
S durant Ortho [[0.99953445+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4311335158716172-5.171269015102367e-13j)
Expectation value gradk*Nab :  (-0.0002620226325307716+2.3568380886061338e-14j)
Expectation value sporb :  (9.14147467057904e-14-2.1533349830497915e-14j)
Fock [[-0.91802105+4.87722836e-11j]]
Kinetic [[1.43075016-4.76653029e-13j]]
Vnuc [[-3.37453882+8.03189155e-24j]]
Coulomb [[2.05153523+5.12464239e-24j]]
Exchange [[1.02576762-4.92489366e-11j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4311335158716172+5.171269015102367e-13j)
Expectation value gradk*Nab :  (-0.0002620226325307716-2.3568380886061338e-14j)
Expectation value sporb :  (1.6024447498483704e-14-3.6287965135504856e-14j)
Fock [[-0.91802105-4.87144622e-11j]]
Kinetic [[1.43075016+5.34474344e-13j]]
Vnuc [[-3.37453882-8.03189155e-24j]]
Coulomb [[2.05153523-5.12464239e-24j]]
Exchange [[1.02576762+4.92489366e-11j]]
Orbital: 0    Norm: [0.99999817]    Update: [0.00022811]    Energy:[-0.91825616+6.88993557e-10j]
Scaled energy (4C Dirac energy) [-0.91823371]
=============Iteration: 4
F= [[-0.91802105-4.87144622e-11j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4310330421922295-5.200046958476922e-13j)
Expectation value gradk*Nab :  (-0.00026051911597895297+2.371703873864434e-14j)
Expectation value sporb :  (9.138295158245189e-14-2.1517369574866386e-14j)
Power Iter, norm of new guess =  (0.9997722427554468+0j) (0.9997722427554468+0j) 0j
orthonorm
S durant Ortho [[0.99977224+0.j]]
Post-Ortho S= [[1.+0.j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4312912068068082-1.472237995902711e-14j)
Expectation value gradk*Nab :  (-0.0002620851900043798+2.0361026678828948e-14j)
Expectation value sporb :  (1.5319242460279975e-14-1.4292718075487608e-13j)
Fock [[-0.91799539-1.71548646e-11j]]
Kinetic [[1.43090776+1.45760698e-13j]]
Vnuc [[-3.37472214-4.91138482e-25j]]
Coulomb [[2.05163799-6.62390726e-25j]]
Exchange [[1.025819+1.73006253e-11j]]
compFock main loop 0
Expectation value (k-1/2)*nabla^2 :  (1.4312912068068082+1.472237995902711e-14j)
Expectation value gradk*Nab :  (-0.0002620851900043798-2.0361026678828948e-14j)
Expectation value sporb :  (3.7664955590040124e-13-5.28110030754466e-14j)
Fock [[-0.91799539+1.73506028e-11j]]
Kinetic [[1.43090776+4.99774856e-14j]]
Vnuc [[-3.37472214+4.91138482e-25j]]
Coulomb [[2.05163799+6.62390726e-25j]]
Exchange [[1.025819-1.73006253e-11j]]
Orbital: 0    Norm: [0.99999996]    Update: [4.20171306e-05]    Energy:[-0.91811896+4.87696427e-11j]
Scaled energy (4C Dirac energy) [-0.91809652]

Ne 1e-4 c=c

start initMolec
Initmolec
S initial guess =  [[1.+0.j]]
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (50.359150303321584+0j)
Expectation value gradk*Nab :  (-0.2459675400625521+0j)
Expectation value sporb :  6.029769739012778e-19j
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (50.359150303321584+0j)
Expectation value gradk*Nab :  (-0.2459675400625521+0j)
Expectation value sporb :  6.029769739012778e-19j
Fock [[-43.86346465-6.0297988e-19j]]
Kinetic [[49.88643466-6.02981707e-19j]]
Vnuc [[-99.99988484+0.j]]
Coulomb [[12.49997105+0.j]]
Exchange [[6.24998553+0.j]]
power Iter
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:170: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu.append(vp.HelmholtzOperator(self.mra, mu, self.prec))  # Initalize the operator
Power Iter Kappa ok
PowerIter derivatives stored
VJkappa ok
Vkphi ok
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:663: ComplexWarning: Casting complex values to real discards the imaginary part
  VzFphi = ((( V_z/(self.c**2) * self.Fock[orb, orb]).crop(self.prec)) * self.phi_prev[orb][-1] ).crop(self.prec)
VzFock ok
VzFphi ok
P I T3 ok
P I T1 ok
P I T2 ok
P I T4 ok
Power Iter, norm of new guess =  (1.00992911953375+0j) (1.00992911953375+0j) 0j
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
init done
scf run
=============Iteration: 0
F= [[-43.86346465-6.0297988e-19j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (48.63128472897783+0j)
Expectation value gradk*Nab :  (-0.23756179792838972+0j)
Expectation value sporb :  -9.296972222350532e-19j
ExpandSol start 
power Iter
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:508: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu[orb] = vp.HelmholtzOperator(self.mra, mu, self.prec)
Power Iter Kappa ok
PowerIter derivatives stored
VJkappa ok
Vkphi ok
VzFock ok
VzFphi ok
P I T3 ok
P I T1 ok
P I T2 ok
P I T4 ok
Power Iter, norm of new guess =  (0.9996777940013115+0j) (0.9996777940013115+0j) 0j
orthonorm
S durant Ortho [[0.99967779+0.j]]
Post-Ortho S= [[1.+0.j]]
ExpandSol millieu
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.85420978232176+0j)
Expectation value gradk*Nab :  (-0.23268911974922987+0j)
Expectation value sporb :  -4.851169660288097e-18j
Fock [[-43.99977909+4.85117412e-18j]]
Kinetic [[47.40894466+4.85119209e-18j]]
Vnuc [[-97.47972401+0.j]]
Coulomb [[12.14200053+0.j]]
Exchange [[6.07100026+0.j]]
end ExpandSol
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.85420978232176+0j)
Expectation value gradk*Nab :  (-0.23268911974922987+0j)
Expectation value sporb :  -4.851169660288097e-18j
Fock [[-43.99977909+4.85117412e-18j]]
Kinetic [[47.40894466+4.85119209e-18j]]
Vnuc [[-97.47972401+0.j]]
Coulomb [[12.14200053+0.j]]
Exchange [[6.07100026+0.j]]
Orbital: 0    Norm: [1.]    Update: [0.0080651]    Energy:[-43.96660032+9.29760853e-19j]
Scaled energy (4C Dirac energy) [-43.91519142]
=============Iteration: 1
F= [[-43.99977909+4.85117412e-18j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.85420978232173+0j)
Expectation value gradk*Nab :  (-0.23268911974922965+0j)
Expectation value sporb :  -5.149973832017955e-18j
ExpandSol start 
power Iter
Power Iter Kappa ok
PowerIter derivatives stored
VJkappa ok
Vkphi ok
VzFock ok
VzFphi ok
P I T3 ok
P I T1 ok
P I T2 ok
P I T4 ok
Power Iter, norm of new guess =  (0.9952355142454792+0j) (0.9952355142454786+0j) (5.763133891193097e-16+0j)
orthonorm
S durant Ortho [[0.99523551+0.j]]
Post-Ortho S= [[1.+0.j]]
ExpandSol millieu
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.49545862381107-7.95928792533055e-21j)
Expectation value gradk*Nab :  (-0.23041253030618256+6.998303621757834e-21j)
Expectation value sporb :  (-5.316577713798092e-14-3.046127521419115e-11j)
Fock [[-44.0123982+3.04608374e-11j]]
Kinetic [[47.05471419+3.04612752e-11j]]
Vnuc [[-97.11392834-3.83211806e-22j]]
Coulomb [[12.09363191+1.27725692e-23j]]
Exchange [[6.04681596+4.37835627e-16j]]
end ExpandSol
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.49545862381107+7.95928792533055e-21j)
Expectation value gradk*Nab :  (-0.23041253030618256-6.998303621757834e-21j)
Expectation value sporb :  (4.691713763623514e-13+1.1167294814341557e-18j)
Fock [[-44.0123982+4.36719977e-16j]]
Kinetic [[47.05471419-1.11569277e-18j]]
Vnuc [[-97.11392834+3.83211806e-22j]]
Coulomb [[12.09363191-1.27725692e-23j]]
Exchange [[6.04681596-4.37835627e-16j]]
Orbital: 0    Norm: [1.]    Update: [0.00356304]    Energy:[-43.99977909+5.15007087e-18j]
Scaled energy (4C Dirac energy) [-43.94829261]
=============Iteration: 2
F= [[-44.0123982+4.36719977e-16j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.49545833285552-1.7691559766249032e-20j)
Expectation value gradk*Nab :  (-0.2304125303155047+7.087066767860288e-21j)
Expectation value sporb :  (-2.0799499028581557e-13-3.0461760486566486e-11j)
ExpandSol start 
power Iter
Power Iter Kappa ok
PowerIter derivatives stored
VJkappa ok
Vkphi ok
VzFock ok
VzFphi ok
P I T3 ok
P I T1 ok
P I T2 ok
P I T4 ok
Power Iter, norm of new guess =  (0.9932148325562474+0j) (0.9932148325562467+0j) (7.640378763409327e-16+0j)
orthonorm
S durant Ortho [[0.99321483+0.j]]
Post-Ortho S= [[1.+0.j]]
ExpandSol millieu
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.32747367896059-1.7513807237914465e-14j)
Expectation value gradk*Nab :  (-0.22928365520058833+2.1724627040831104e-14j)
Expectation value sporb :  (8.676175285508057e-14-3.531602709587967e-11j)
Fock [[-44.01767479+3.70405419e-11j]]
Kinetic [[46.88894756+3.53158798e-11j]]
Vnuc [[-96.94236113-9.62566026e-24j]]
Coulomb [[12.07147756+3.99413588e-24j]]
Exchange [[6.03573878-1.72466206e-12j]]
end ExpandSol
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.32747367896059+1.7513807237914465e-14j)
Expectation value gradk*Nab :  (-0.22928365520058833-2.1724627040831104e-14j)
Expectation value sporb :  (2.574491214835327e-13-1.7448236491224006e-11j)
Fock [[-44.01767479+1.57237217e-11j]]
Kinetic [[46.88894756+1.74483837e-11j]]
Vnuc [[-96.94236113+9.62566026e-24j]]
Coulomb [[12.07147756-3.99413588e-24j]]
Exchange [[6.03573878+1.72466206e-12j]]
Orbital: 0    Norm: [1.]    Update: [0.0016141]    Energy:[-44.01239849+3.04613226e-11j]
Scaled energy (4C Dirac energy) [-43.96088249]
=============Iteration: 3
F= [[-44.01767479+1.57237217e-11j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.327473386942714-3.3521746496420916e-14j)
Expectation value gradk*Nab :  (-0.22928365520999258+2.1694232244653543e-14j)
Expectation value sporb :  (7.862960428221683e-14-3.53165801461125e-11j)
ExpandSol start 
power Iter
Power Iter Kappa ok
PowerIter derivatives stored
VJkappa ok
Vkphi ok
VzFock ok
VzFphi ok
P I T3 ok
P I T1 ok
P I T2 ok
P I T4 ok
Power Iter, norm of new guess =  (0.9922767846524574+0j) (0.9922767846524565+0j) (9.262690246191519e-16+0j)
orthonorm
S durant Ortho [[0.99227678+0.j]]
Post-Ortho S= [[1.+0.j]]
ExpandSol millieu
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.2483327278205-2.983372766201752e-14j)
Expectation value gradk*Nab :  (-0.22873618581844377+2.0497117520457334e-14j)
Expectation value sporb :  (9.318604992486888e-14-4.611302941107402e-11j)
Fock [[-44.02000707+4.65740225e-11j]]
Kinetic [[46.81087875+4.61009402e-11j]]
Vnuc [[-96.86148421-2.27046526e-23j]]
Coulomb [[12.06119678-1.87165947e-24j]]
Exchange [[6.03059839-4.73082254e-13j]]
end ExpandSol
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.2483327278205+2.983372766201752e-14j)
Expectation value gradk*Nab :  (-0.22873618581844377-2.0497117520457334e-14j)
Expectation value sporb :  (3.710883456138889e-13-2.1106066885915776e-11j)
Fock [[-44.02000707+2.06450738e-11j]]
Kinetic [[46.81087875+2.11181561e-11j]]
Vnuc [[-96.86148421+2.27046526e-23j]]
Coulomb [[12.06119678+1.87165947e-24j]]
Exchange [[6.03059839+4.73082254e-13j]]
Orbital: 0    Norm: [1.]    Update: [0.00074298]    Energy:[-44.01767508+3.70252116e-11j]
Scaled energy (4C Dirac energy) [-43.96614674]
=============Iteration: 4
F= [[-44.02000707+2.06450738e-11j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.2483324350937-3.1837876553675535e-14j)
Expectation value gradk*Nab :  (-0.22873618582787075+1.9747120798560615e-14j)
Expectation value sporb :  (5.222236696386945e-14-4.611360607533707e-11j)
ExpandSol start 
power Iter
Power Iter Kappa ok
PowerIter derivatives stored
VJkappa ok
Vkphi ok
VzFock ok
VzFphi ok
P I T3 ok
P I T1 ok
P I T2 ok
P I T4 ok
Power Iter, norm of new guess =  (0.9918368357970893+0j) (0.9918368357970883+0j) (9.953237816585454e-16+0j)
orthonorm
S durant Ortho [[0.99183684+0.j]]
Post-Ortho S= [[1.+0.j]]
ExpandSol millieu
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.21092890139028-2.2206705635950607e-14j)
Expectation value gradk*Nab :  (-0.2284725525728069+2.740985553401399e-14j)
Expectation value sporb :  (1.0667500099869511e-13-5.004635242995724e-11j)
Fock [[-44.02107091+5.22170469e-11j]]
Kinetic [[46.77399006+5.00462225e-11j]]
Vnuc [[-96.82325489+6.43456115e-23j]]
Coulomb [[12.05638784-3.96687221e-23j]]
Exchange [[6.02819392-2.17082441e-12j]]
end ExpandSol
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.21092890139028+2.2206705635950607e-14j)
Expectation value gradk*Nab :  (-0.2284725525728069-2.740985553401399e-14j)
Expectation value sporb :  (3.93510454275631e-13-2.7195342936232078e-11j)
Fock [[-44.02107091+2.50246484e-11j]]
Kinetic [[46.77399006+2.71954729e-11j]]
Vnuc [[-96.82325489-6.43456115e-23j]]
Coulomb [[12.05638784+3.96687221e-23j]]
Exchange [[6.02819392+2.17082441e-12j]]
Orbital: 0    Norm: [1.]    Update: [0.00034554]    Energy:[-44.02000736+4.65714906e-11j]
Scaled energy (4C Dirac energy) [-43.96847356]
=============Iteration: 5
F= [[-44.02107091+2.50246484e-11j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.21092860517594-4.690326152372335e-14j)
Expectation value gradk*Nab :  (-0.2284725525822414+2.7418464716976426e-14j)
Expectation value sporb :  (8.205598183399235e-14-5.0046931897384785e-11j)
ExpandSol start 
power Iter
Power Iter Kappa ok
PowerIter derivatives stored
VJkappa ok
Vkphi ok
VzFock ok
VzFphi ok
P I T3 ok
P I T1 ok
P I T2 ok
P I T4 ok
Power Iter, norm of new guess =  (0.9916294136580908+0j) (0.9916294136580898+0j) (1.036701044425932e-15+0j)
orthonorm
S durant Ortho [[0.99162941+0.j]]
Post-Ortho S= [[1.+0.j]]
ExpandSol millieu
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.19322408845887-2.5149426708769033e-14j)
Expectation value gradk*Nab :  (-0.22834626575864234+3.045562671939371e-14j)
Expectation value sporb :  (1.0815464425867342e-13-5.325521538073693e-11j)
Fock [[-44.0215643+5.4242719e-11j]]
Kinetic [[46.75653168+5.32561158e-11j]]
Vnuc [[-96.80515973+1.74145783e-22j]]
Coulomb [[12.0541275-1.68324349e-24j]]
Exchange [[6.02706375-9.86603134e-13j]]
end ExpandSol
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.19322408845887+2.5149426708769033e-14j)
Expectation value gradk*Nab :  (-0.22834626575864234-3.045562671939371e-14j)
Expectation value sporb :  (4.3647606275083573e-13-2.9041631242732327e-11j)
Fock [[-44.0215643+2.80541277e-11j]]
Kinetic [[46.75653168+2.90407308e-11j]]
Vnuc [[-96.80515973-1.74145783e-22j]]
Coulomb [[12.0541275+1.68324349e-24j]]
Exchange [[6.02706375+9.86603134e-13j]]
Orbital: 0    Norm: [1.]    Update: [0.00016176]    Energy:[-44.02107121+5.21933332e-11j]
Scaled energy (4C Dirac energy) [-43.96953492]
=============Iteration: 6
F= [[-44.0215643+2.80541277e-11j]]
S= [[1.+0.j]]
orthonorm
S durant Ortho [[1.+0.j]]
Post-Ortho S= [[1.+0.j]]
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.19322379204027-4.577050716255112e-14j)
Expectation value gradk*Nab :  (-0.2283462657680822+2.9513179882228324e-14j)
Expectation value sporb :  (7.126157017686887e-14-5.3255811753873984e-11j)
ExpandSol start 
power Iter
Power Iter Kappa ok
PowerIter derivatives stored
VJkappa ok
Vkphi ok
VzFock ok
VzFphi ok
P I T3 ok
P I T1 ok
P I T2 ok
P I T4 ok
Power Iter, norm of new guess =  (0.9915313681806103+0j) (0.9915313681806092+0j) (1.0559615596933217e-15+0j)
orthonorm
S durant Ortho [[0.99153137+0.j]]
Post-Ortho S= [[1.+0.j]]
ExpandSol millieu
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.18483699561149-3.851298739428764e-13j)
Expectation value gradk*Nab :  (-0.2282859772505089-3.932125382122883e-14j)
Expectation value sporb :  (1.0875551071039002e-13-5.4430203234325056e-11j)
Fock [[-44.02179532+5.32718939e-11j]]
Kinetic [[46.74826214+5.40180177e-11j]]
Vnuc [[-96.79658833-1.08927723e-22j]]
Coulomb [[12.05306174-1.88976221e-23j]]
Exchange [[6.02653087+7.46123802e-13j]]
end ExpandSol
compCoulomb
compFock main loop 0
compExch
Expectation value (k-1/2)*nabla^2 :  (47.18483699561149+3.851298739428764e-13j)
Expectation value gradk*Nab :  (-0.2282859772505089+3.932125382122883e-14j)
Expectation value sporb :  (4.56073709347883e-13-3.068177862800643e-11j)
Fock [[-44.02179532+3.1840088e-11j]]
Kinetic [[46.74826214+3.10939642e-11j]]
Vnuc [[-96.79658833+1.08927723e-22j]]
Coulomb [[12.05306174+1.88976221e-23j]]
Exchange [[6.02653087-7.46123802e-13j]]
Orbital: 0    Norm: [1.]    Update: [7.60551266e-05]    Energy:[-44.0215646+5.42215128e-11j]
Scaled energy (4C Dirac energy) [-43.97002715]

LiH 1e-4 c=c non scalar

start initMolec
Fock [[-3.12551422-7.78524619e-08j  0.00697997-5.53988109e-10j]
 [ 0.00697862-7.80642169e-09j -0.7829294 -7.10728536e-10j]]
Kinetic [[3.60417535-7.78524619e-08j 0.49236947-7.80642169e-09j]
 [0.49237252-5.53988109e-10j 0.39084441-7.10728536e-10j]]
Vnuc [[-8.3770787 +0.j -0.5745742 +0.j]
 [-0.5745742 +0.j -1.85352593+0.j]]
Coulomb [[3.29477826+0.j 0.1783667 +0.j]
 [0.1783667 +0.j 0.68880935+0.j]]
Exchange [[1.64738913+0.j 0.08918335+0.j]
 [0.08918506+0.j 0.00905723+0.j]]
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:170: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu.append(vp.HelmholtzOperator(self.mra, mu, self.prec))  # Initalize the operator
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:675: ComplexWarning: Casting complex values to real discards the imaginary part
  VzFphi = ((( V_z/(self.c**2) * self.Fock[orb, orb]).crop(self.prec)) * self.phi_prev[orb][-1] ).crop(self.prec)
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in 0.9999999999999988 (0.009117505616230571-3.758661614004517e-09j) (0.15070025617856683+5.702995833706879e-10j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.float64'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fc3c734a5b0>
Orthonormalise GS (-1.524006540811005e-12-6.7872656600791595e-09j) (-4.0549161250957866e-16+1.245904563138872e-09j) (4.0549161250957866e-16-1.245904563138872e-09j)
init done
scf run
=============Iteration: 0
F= [[-3.12551422-7.78524619e-08j  0.00697997-5.53988109e-10j]
 [ 0.00697862-7.80642169e-09j -0.7829294 -7.10728536e-10j]]
CompOverlap 4
S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+2.07844126e-10j
  -1.52400654e-12-6.78726566e-09j -6.53992919e-14+2.21831807e-09j]
 [ 0.00000000e+00-2.07844126e-10j  1.00000000e+00+0.00000000e+00j
  -4.05491613e-16-1.24590456e-09j -1.52400654e-12+6.78726566e-09j]
 [-1.52400654e-12+6.78726566e-09j -4.05491613e-16+1.24590456e-09j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00-5.88756766e-10j]
 [-6.53992919e-14-2.21831807e-09j -1.52400654e-12-6.78726566e-09j
   0.00000000e+00+5.88756766e-10j  1.00000000e+00+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in 1.0000000000000004 (-1.524006540811005e-12+6.7872656600791595e-09j) (-4.0549161250957866e-16+1.245904563138872e-09j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.float64'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fc3c734a730>
Orthonormalise GS (1.5239926088130885e-12+1.439175568202644e-08j) (6.580630138031118e-14-9.72413508872921e-10j) (-6.580630138031118e-14+9.72413508872921e-10j)
CompOverlap 4
ExpandSol initial S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+2.07844126e-10j
   1.52399261e-12+1.43917557e-08j -2.10335221e-17-8.83118701e-22j]
 [ 0.00000000e+00-2.07844126e-10j  1.00000000e+00+0.00000000e+00j
   6.58063014e-14+9.72413509e-10j  1.52399261e-12-1.43917557e-08j]
 [ 1.52399261e-12-1.43917557e-08j  6.58063014e-14-9.72413509e-10j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00+5.88756766e-10j]
 [-2.10335221e-17+8.83118701e-22j  1.52399261e-12+1.43917557e-08j
   0.00000000e+00-5.88756766e-10j  1.00000000e+00+0.00000000e+00j]]
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:508: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu[orb] = vp.HelmholtzOperator(self.mra, mu, self.prec)
CompOverlap 4
ExpandSol pre orthonorm S= [[ 3.99650937e+00+0.00000000e+00j  0.00000000e+00-2.64789137e-09j
   3.30461804e-02+2.05926054e-08j -5.17509725e-01+4.62590091e-09j]
 [ 0.00000000e+00+2.64789137e-09j  3.99650937e+00+0.00000000e+00j
   5.17509725e-01+5.15726845e-10j  3.30461804e-02-2.05926054e-08j]
 [ 3.30461804e-02-2.05926054e-08j  5.17509725e-01-5.15726845e-10j
   4.09948894e+00+0.00000000e+00j -1.11022302e-16+6.84961855e-09j]
 [-5.17509725e-01-4.62590091e-09j  3.30461804e-02+2.05926054e-08j
  -1.11022302e-16-6.84961855e-09j  4.09948894e+00+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in 1.999127152561442 (0.03304618043069713-2.0592605362253202e-08j) (0.5175097252532335-5.157268453458235e-10j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.float64'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fc3c734a820>
Orthonormalise GS (-1.5415777107532364e-12+1.308708243807767e-09j) (-2.9516796992701266e-13-1.280822688341498e-09j) (2.9516796992701266e-13+1.280822688341498e-09j)
CompOverlap 4
ExpandSol milieu S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00-6.62551023e-10j
  -1.54157771e-12+1.30870824e-09j -2.83540552e-14-2.84784248e-09j]
 [ 0.00000000e+00+6.62551023e-10j  1.00000000e+00+0.00000000e+00j
  -2.95167970e-13+1.28082269e-09j -1.54157771e-12-1.30870824e-09j]
 [-1.54157771e-12-1.30870824e-09j -2.95167970e-13-1.28082269e-09j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00-3.36772086e-09j]
 [-2.83540552e-14+2.84784248e-09j -1.54157771e-12+1.30870824e-09j
   0.00000000e+00+3.36772086e-09j  1.00000000e+00+0.00000000e+00j]]
Solution syst linéaire  0  =  [-0.78372807-1.68345434e-05j]
Solution syst linéaire  1  =  [-0.6585867-3.52329079e-08j]
CompOverlap 4
ExpandSol final S= [[ 1.        +0.00000000e+00j  0.        +1.79636529e-05j
  -0.07463297-1.65766475e-06j  0.44422205+3.45028586e-05j]
 [ 0.        -1.79636529e-05j  1.        +0.00000000e+00j
  -0.44422205+3.45396351e-05j -0.07463297+1.65766475e-06j]
 [-0.07463297+1.65766475e-06j -0.44422205-3.45396351e-05j
   1.        +0.00000000e+00j  0.        +2.85667842e-08j]
 [ 0.44422205-3.45028586e-05j -0.07463297-1.65766475e-06j
   0.        -2.85667842e-08j  1.        +0.00000000e+00j]]
Fock [[-3.11736311-2.54534725e-04j  0.24133271-3.37829473e-05j]
 [ 0.19974643-5.19369591e-06j -0.33310545+3.01308832e-05j]]
Kinetic [[ 3.63597781-5.54727183e-04j -0.06138614-1.67161970e-05j]
 [-0.06137689-1.66521094e-05j  2.28914528+1.92832700e-08j]]
Vnuc [[-8.41030887-3.25525350e-19j  0.38861369-1.24869965e-05j]
 [ 0.38861369+1.24869965e-05j -4.56106029+1.88597180e-21j]]
Coulomb [[ 3.31393578+2.16774260e-19j -0.17180764+2.77908365e-06j]
 [-0.17180764-2.77908365e-06j  1.55759089+2.64697796e-22j]]
Exchange [[ 1.65696789+1.75441699e-19j -0.04432652-1.82921270e-06j]
 [-0.08590355+2.68387508e-05j -0.38121868-3.01115999e-05j]]
Fock [[-3.11736311-2.54534725e-04j  0.24133271-3.37829473e-05j]
 [ 0.19974643-5.19369591e-06j -0.33310545+3.01308832e-05j]]
Kinetic [[ 3.63597781-5.54727183e-04j -0.06138614-1.67161970e-05j]
 [-0.06137689-1.66521094e-05j  2.28914528+1.92832700e-08j]]
Vnuc [[-8.41030887-3.25525350e-19j  0.38861369-1.24869965e-05j]
 [ 0.38861369+1.24869965e-05j -4.56106029+1.88597180e-21j]]
Coulomb [[ 3.31393578+2.16774260e-19j -0.17180764+2.77908365e-06j]
 [-0.17180764-2.77908365e-06j  1.55759089+2.64697796e-22j]]
Exchange [[ 1.65696789+1.75441699e-19j -0.04432652-1.82921270e-06j]
 [-0.08590355+2.68387508e-05j -0.38121868-3.01115999e-05j]]
Orbital: 0    Norm: [0.21870783 0.4386203 ]    Update: [0.78312283 0.68840806]    Energy:[-3.12525418-1.15602279e-08j -0.79160869+1.26428217e-10j]
Orbital: 1    Norm: [0.21870783 0.4386203 ]    Update: [0.78312283 0.68840806]    Energy:[-3.12525418-1.15602279e-08j -0.79160869+1.26428217e-10j]
=============Iteration: 1
F= [[-3.11736311-2.54534725e-04j  0.24133271-3.37829473e-05j]
 [ 0.19974643-5.19369591e-06j -0.33310545+3.01308832e-05j]]
CompOverlap 4
S= [[ 1.        +0.00000000e+00j  0.        -1.79636529e-05j
  -0.07463297+1.65766475e-06j  0.44422205-3.45028586e-05j]
 [ 0.        +1.79636529e-05j  1.        +0.00000000e+00j
  -0.44422205-3.45396351e-05j -0.07463297-1.65766475e-06j]
 [-0.07463297-1.65766475e-06j -0.44422205+3.45396351e-05j
   1.        +0.00000000e+00j  0.        -2.85667842e-08j]
 [ 0.44422205+3.45028586e-05j -0.07463297+1.65766475e-06j
   0.        +2.85667842e-08j  1.        +0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in 1.0 (-0.07463296903389312-1.6576647529170219e-06j) (-0.44422205322775377+3.453963512747701e-05j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.float64'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fc3c734ab20>
Orthonormalise GS (-1.9105501922276474e-09-3.7800968676685777e-06j) (9.260226266349925e-12+7.733230698449002e-05j) (-9.260226266349925e-12-7.733230698449002e-05j)
CompOverlap 4
ExpandSol initial S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00-1.79636529e-05j
  -1.91055019e-09-3.78009687e-06j  1.18133612e-08+7.75086174e-05j]
 [ 0.00000000e+00+1.79636529e-05j  1.00000000e+00+0.00000000e+00j
   9.26022627e-12-7.73323070e-05j -1.91055019e-09+3.78009687e-06j]
 [-1.91055019e-09+3.78009687e-06j  9.26022627e-12+7.73323070e-05j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00+1.70905393e-08j]
 [ 1.18133612e-08-7.75086174e-05j -1.91055019e-09-3.78009687e-06j
   0.00000000e+00-1.70905393e-08j  1.00000000e+00+0.00000000e+00j]]
CompOverlap 4
ExpandSol pre orthonorm S= [[ 4.00845281e+00+0.00000000e+00j  1.66533454e-16+2.67690755e-05j
  -8.55883204e-02-1.62602101e-05j -5.41594530e-01-4.21984964e-04j]
 [ 1.66533454e-16-2.67690755e-05j  4.00845281e+00+0.00000000e+00j
   5.41594556e-01+4.01495263e-04j -8.55883204e-02+1.62602101e-05j]
 [-8.55883204e-02+1.62602101e-05j  5.41594556e-01-4.01495263e-04j
   5.42435348e+00+0.00000000e+00j  0.00000000e+00-1.02698629e-04j]
 [-5.41594530e-01+4.21984964e-04j -8.55883204e-02-1.62602101e-05j
   0.00000000e+00+1.02698629e-04j  5.42435348e+00+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in 2.002112088061502 (-0.08558832044090402+1.6260210101539062e-05j) (0.5415945559648603-0.00040149526344809475j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.float64'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fc3c734ab20>
Orthonormalise GS (-4.692138257252586e-10+8.633580477890505e-06j) (-5.601717918083532e-09+4.424831616064978e-06j) (5.601717918083532e-09-4.424831616064978e-06j)
CompOverlap 4
ExpandSol milieu S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+6.67815658e-06j
  -4.69213826e-10+8.63358048e-06j  5.39808935e-09+7.55186348e-06j]
 [ 0.00000000e+00-6.67815658e-06j  1.00000000e+00+0.00000000e+00j
  -5.60171792e-09-4.42483162e-06j -4.69213826e-10-8.63358048e-06j]
 [-4.69213826e-10-8.63358048e-06j -5.60171792e-09+4.42483162e-06j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00+1.48886721e-05j]
 [ 5.39808935e-09-7.55186348e-06j -4.69213826e-10+8.63358048e-06j
   0.00000000e+00-1.48886721e-05j  1.00000000e+00+0.00000000e+00j]]
Solution syst linéaire  0  =  [0.06641586+0.00062386j 0.37100864-0.00105355j]
Solution syst linéaire  1  =  [0.01897293+3.31119663e-06j 0.44070563+4.01723910e-05j]
CompOverlap 4
ExpandSol final S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00-1.15671335e-05j
   6.55310900e-04-4.73742033e-06j -5.49407219e-03+4.89869311e-07j]
 [ 0.00000000e+00+1.15671335e-05j  1.00000000e+00+0.00000000e+00j
   5.49406928e-03+5.38323734e-06j  6.55310900e-04+4.73742033e-06j]
 [ 6.55310900e-04+4.73742033e-06j  5.49406928e-03-5.38323734e-06j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00-2.50031293e-05j]
 [-5.49407219e-03-4.89869311e-07j  6.55310900e-04-4.73742033e-06j
   0.00000000e+00+2.50031293e-05j  1.00000000e+00+0.00000000e+00j]]
Fock [[-3.12441645+1.98868524e-03j  0.00616913-3.96173658e-05j]
 [ 0.0131743 +1.02429097e-04j -0.75762391-1.18092110e-06j]]
Kinetic [[3.61911744+4.38998072e-03j 0.37135212+2.24188698e-04j]
 [0.37137261+2.33665880e-04j 0.68216377-1.66850098e-09j]]
Vnuc [[-8.39593624-1.75844040e-18j -0.43246048+2.90046838e-04j]
 [-0.43246048-2.90046838e-04j -2.20445556-7.62329653e-21j]]
Coulomb [[3.30479697+4.28598671e-19j 0.13451408-9.35959316e-05j]
 [0.13451408+9.35959316e-05j 0.77336838+2.03287907e-20j]]
Exchange [[1.65239849+1.94817578e-20j 0.06023141-5.06766669e-05j]
 [0.06725708+7.68323399e-05j 0.00870049+1.17925260e-06j]]
Fock [[-3.12441645+1.98868524e-03j  0.00616913-3.96173658e-05j]
 [ 0.0131743 +1.02429097e-04j -0.75762391-1.18092110e-06j]]
Kinetic [[3.61911744+4.38998072e-03j 0.37135212+2.24188698e-04j]
 [0.37137261+2.33665880e-04j 0.68216377-1.66850098e-09j]]
Vnuc [[-8.39593624-1.75844040e-18j -0.43246048+2.90046838e-04j]
 [-0.43246048-2.90046838e-04j -2.20445556-7.62329653e-21j]]
Coulomb [[3.30479697+4.28598671e-19j 0.13451408-9.35959316e-05j]
 [0.13451408+9.35959316e-05j 0.77336838+2.03287907e-20j]]
Exchange [[1.65239849+1.94817578e-20j 0.06023141-5.06766669e-05j]
 [0.06725708+7.68323399e-05j 0.00870049+1.17925260e-06j]]
Orbital: 0    Norm: [1.06492582 1.00929438]    Update: [0.08488243 0.19853858]    Energy:[-3.11736311-2.52250096e-04j -0.65991936+3.31310911e-05j]
Orbital: 1    Norm: [1.06492582 1.00929438]    Update: [0.08488243 0.19853858]    Energy:[-3.11736311-2.52250096e-04j -0.65991936+3.31310911e-05j]
=============Iteration: 2
F= [[-3.12441645+1.98868524e-03j  0.00616913-3.96173658e-05j]
 [ 0.0131743 +1.02429097e-04j -0.75762391-1.18092110e-06j]]
CompOverlap 4
S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+1.15671335e-05j
   6.55310900e-04+4.73742033e-06j -5.49407219e-03-4.89869311e-07j]
 [ 0.00000000e+00-1.15671335e-05j  1.00000000e+00+0.00000000e+00j
   5.49406928e-03-5.38323734e-06j  6.55310900e-04-4.73742033e-06j]
 [ 6.55310900e-04-4.73742033e-06j  5.49406928e-03+5.38323734e-06j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00+2.50031293e-05j]
 [-5.49407219e-03+4.89869311e-07j  6.55310900e-04+4.73742033e-06j
   0.00000000e+00-2.50031293e-05j  1.00000000e+00+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in 1.000000000000001 (0.0006553108997501003-4.737420334283072e-06j) (0.0054940692764892255+5.383237342021669e-06j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.float64'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fc3c7337b20>
Orthonormalise GS (1.0923035697457933e-08-8.587924799136543e-06j) (2.9142231453525846e-09+5.873196555563826e-06j) (-2.9142231453525846e-09-5.873196555563826e-06j)
CompOverlap 4
ExpandSol initial S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+1.15671335e-05j
   1.09230357e-08-8.58792480e-06j -1.03792257e-08+6.60858230e-06j]
 [ 0.00000000e+00-1.15671335e-05j  1.00000000e+00+0.00000000e+00j
   2.91422315e-09-5.87319656e-06j  1.09230357e-08+8.58792480e-06j]
 [ 1.09230357e-08+8.58792480e-06j  2.91422315e-09+5.87319656e-06j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00-2.49682867e-05j]
 [-1.03792257e-08-6.60858230e-06j  1.09230357e-08-8.58792480e-06j
   0.00000000e+00+2.49682867e-05j  1.00000000e+00+0.00000000e+00j]]
CompOverlap 4
ExpandSol pre orthonorm S= [[ 3.99908845+0.00000000e+00j  0.        +4.34404439e-06j
  -0.00653076+2.22795693e-04j -0.50760242-3.70210484e-04j]
 [ 0.        -4.34404439e-06j  3.99908845+0.00000000e+00j
   0.5076022 +5.46264832e-04j -0.00653076-2.22795693e-04j]
 [-0.00653076-2.22795693e-04j  0.5076022 -5.46264832e-04j
   4.57517872+0.00000000e+00j  0.        +1.73506019e-05j]
 [-0.50760242+3.70210484e-04j -0.00653076+2.22795693e-04j
   0.        -1.73506019e-05j  4.57517872+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in 1.999772100738015 (-0.006530763049632363-0.00022279569342518627j) (0.5076022038320438-0.0005462648316114256j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.float64'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fc3c734a040>
Orthonormalise GS (-5.2400866689417147e-08-0.00010582940699007301j) (5.123621463687042e-08-4.1451708230994135e-05j) (-5.123621463687042e-08+4.1451708230994135e-05j)
CompOverlap 4
ExpandSol milieu S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+1.08625864e-06j
  -5.24008667e-08-1.05829407e-04j -8.37964558e-08-6.21260772e-05j]
 [ 0.00000000e+00-1.08625864e-06j  1.00000000e+00+0.00000000e+00j
   5.12362146e-08+4.14517082e-05j -5.24008667e-08+1.05829407e-04j]
 [-5.24008667e-08+1.05829407e-04j  5.12362146e-08-4.14517082e-05j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00+8.53099573e-06j]
 [-8.37964558e-08+6.21260772e-05j -5.24008667e-08-1.05829407e-04j
   0.00000000e+00-8.53099573e-06j  1.00000000e+00+0.00000000e+00j]]
Solution syst linéaire  0  =  [ 8.23316743e-03+0.01379744j -1.04435295e+01+0.68859244j
 -5.13185675e+00+0.3238523j ]
Solution syst linéaire  1  =  [ 0.05527032-2.98378616e-05j -1.84693222+7.66653523e-03j
 -1.44696713+5.01663367e-03j]
CompOverlap 4
ExpandSol final S= [[ 1.        +0.00000000e+00j  0.        -5.56804364e-02j
   0.00611867+7.19019960e-04j -0.01886456-8.28413540e-05j]
 [ 0.        +5.56804364e-02j  1.        +0.00000000e+00j
   0.01889018-1.38999817e-04j  0.00611867-7.19019960e-04j]
 [ 0.00611867-7.19019960e-04j  0.01889018+1.38999817e-04j
   1.        +0.00000000e+00j  0.        -1.20456806e-03j]
 [-0.01886456+8.28413540e-05j  0.00611867+7.19019960e-04j
   0.        +1.20456806e-03j  1.        +0.00000000e+00j]]
Fock [[-3.12499234-0.01454736j -0.01746175+0.02707215j]
 [-0.01503471+0.00574559j -0.75107828-0.00037815j]]
Kinetic [[3.60475237-3.69411666e-02j 0.02215135+2.52249543e-02j]
 [0.02211246+2.11707597e-02j 0.70698097-4.06481308e-09j]]
Vnuc [[-8.38396798-1.38777878e-16j -0.06216881+2.07341242e-02j]
 [-0.06216881-2.07341242e-02j -2.23878519+6.77626358e-19j]]
Coulomb [[3.29204163-3.81639165e-17j 0.04515896-4.39859886e-03j]
 [0.04515896+4.39859886e-03j 0.79411364-2.77826807e-19j]]
Exchange [[1.64602082+1.56125113e-17j 0.02007005-2.14977124e-03j]
 [0.02256437-2.22369110e-02j 0.0133877 +3.78143941e-04j]]
Fock [[-3.12499234-0.01454736j -0.01746175+0.02707215j]
 [-0.01503471+0.00574559j -0.75107828-0.00037815j]]
Kinetic [[3.60475237-3.69411666e-02j 0.02215135+2.52249543e-02j]
 [0.02211246+2.11707597e-02j 0.70698097-4.06481308e-09j]]
Vnuc [[-8.38396798-1.38777878e-16j -0.06216881+2.07341242e-02j]
 [-0.06216881-2.07341242e-02j -2.23878519+6.77626358e-19j]]
Coulomb [[3.29204163-3.81639165e-17j 0.04515896-4.39859886e-03j]
 [0.04515896+4.39859886e-03j 0.79411364-2.77826807e-19j]]
Exchange [[1.64602082+1.56125113e-17j 0.02007005-2.14977124e-03j]
 [0.02256437-2.22369110e-02j 0.0133877 +3.78143941e-04j]]
Orbital: 0    Norm: [1.11327538 1.09126446]    Update: [0.43249462 0.20110594]    Energy:[-3.12441645+1.98869784e-03j -0.75426237-1.41370018e-06j]
Orbital: 1    Norm: [1.11327538 1.09126446]    Update: [0.43249462 0.20110594]    Energy:[-3.12441645+1.98869784e-03j -0.75426237-1.41370018e-06j]
=============Iteration: 3
F= [[-3.12499234-0.01454736j -0.01746175+0.02707215j]
 [-0.01503471+0.00574559j -0.75107828-0.00037815j]]
CompOverlap 4
S= [[ 1.        +0.00000000e+00j  0.        +5.56804364e-02j
   0.00611867-7.19019960e-04j -0.01886456+8.28413540e-05j]
 [ 0.        -5.56804364e-02j  1.        +0.00000000e+00j
   0.01889018+1.38999817e-04j  0.00611867+7.19019960e-04j]
 [ 0.00611867+7.19019960e-04j  0.01889018-1.38999817e-04j
   1.        +0.00000000e+00j  0.        +1.20456806e-03j]
 [-0.01886456-8.28413540e-05j  0.00611867-7.19019960e-04j
   0.        -1.20456806e-03j  1.        +0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
Norm Phi_in 0.9999999999999993 (0.006118671634532433+0.0007190199598675871j) (0.01889017634288006-0.00013899981673688662j)
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'numpy.float64'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> None
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fc3c7378670>
Orthonormalise GS (1.2044700019039922e-05+0.0020125675165204985j) (-2.562404893157659e-05-0.00022188463281375202j) (2.562404893157659e-05+0.00022188463281375202j)

LiH 1e-4 c=c GS(norme = compSqNorm)

start initMolec
Fock [[-3.12551422-7.78524619e-08j  0.00697997-5.53988109e-10j]
 [ 0.00697862-7.80642169e-09j -0.7829294 -7.10728536e-10j]]
Kinetic [[3.60417535-7.78524619e-08j 0.49236947-7.80642169e-09j]
 [0.49237252-5.53988109e-10j 0.39084441-7.10728536e-10j]]
Vnuc [[-8.3770787 +0.j -0.5745742 +0.j]
 [-0.5745742 +0.j -1.85352593+0.j]]
Coulomb [[3.29477826+0.j 0.1783667 +0.j]
 [0.1783667 +0.j 0.68880935+0.j]]
Exchange [[1.64738913+0.j 0.08918335+0.j]
 [0.08918506+0.j 0.00905723+0.j]]
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:170: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu.append(vp.HelmholtzOperator(self.mra, mu, self.prec))  # Initalize the operator
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:675: ComplexWarning: Casting complex values to real discards the imaginary part
  VzFphi = ((( V_z/(self.c**2) * self.Fock[orb, orb]).crop(self.prec)) * self.phi_prev[orb][-1] ).crop(self.prec)
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0110daa60>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110da580>
Orthonormalise GS (-1.5240102474271822e-12-6.7872656600791595e-09j) (-5.730008481585891e-16+1.2459045631388692e-09j) (5.730008481585891e-16-1.2459045631388692e-09j)
init done
scf run
=============Iteration: 0
F= [[-3.12551422-7.78524619e-08j  0.00697997-5.53988109e-10j]
 [ 0.00697862-7.80642169e-09j -0.7829294 -7.10728536e-10j]]
CompOverlap 4
S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+2.07844126e-10j
  -1.52401025e-12-6.78726566e-09j -6.52317826e-14+2.21831807e-09j]
 [ 0.00000000e+00-2.07844126e-10j  1.00000000e+00+0.00000000e+00j
  -5.73000848e-16-1.24590456e-09j -1.52401025e-12+6.78726566e-09j]
 [-1.52401025e-12+6.78726566e-09j -5.73000848e-16+1.24590456e-09j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00-5.88756766e-10j]
 [-6.52317826e-14-2.21831807e-09j -1.52401025e-12-6.78726566e-09j
   0.00000000e+00+5.88756766e-10j  1.00000000e+00+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0110dae50>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110dabb0>
Orthonormalise GS (1.5239812517953318e-12+1.4391755682026445e-08j) (6.579925406619003e-14-9.724135088729181e-10j) (-6.579925406619003e-14+9.724135088729181e-10j)
CompOverlap 4
ExpandSol initial S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+2.07844126e-10j
   1.52398125e-12+1.43917557e-08j -1.39862080e-17-8.77328437e-22j]
 [ 0.00000000e+00-2.07844126e-10j  1.00000000e+00+0.00000000e+00j
   6.57992541e-14+9.72413509e-10j  1.52398125e-12-1.43917557e-08j]
 [ 1.52398125e-12-1.43917557e-08j  6.57992541e-14-9.72413509e-10j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00+5.88756766e-10j]
 [-1.39862080e-17+8.77328437e-22j  1.52398125e-12+1.43917557e-08j
   0.00000000e+00-5.88756766e-10j  1.00000000e+00+0.00000000e+00j]]
/home/qpitto/Tests_KAIN/ZORA/response/src/scfsolv.py:508: ComplexWarning: Casting complex values to real discards the imaginary part
  self.G_mu[orb] = vp.HelmholtzOperator(self.mra, mu, self.prec)
CompOverlap 4
ExpandSol pre orthonorm S= [[ 3.99650937e+00+0.00000000e+00j  2.77555756e-17-2.64789137e-09j
   3.30461804e-02+2.05926054e-08j -5.17509725e-01+4.62590091e-09j]
 [ 2.77555756e-17+2.64789137e-09j  3.99650937e+00+0.00000000e+00j
   5.17509725e-01+5.15726845e-10j  3.30461804e-02-2.05926054e-08j]
 [ 3.30461804e-02-2.05926054e-08j  5.17509725e-01-5.15726845e-10j
   4.09948894e+00+0.00000000e+00j -5.55111512e-17+6.84961855e-09j]
 [-5.17509725e-01-4.62590091e-09j  3.30461804e-02+2.05926054e-08j
  -5.55111512e-17-6.84961855e-09j  4.09948894e+00+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0110dad30>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110da250>
Orthonormalise GS (0.0041056958727646+3.816514014954747e-09j) (0.06429600989229149-1.2140871803594866e-09j) (-0.06429600989229149+1.2140871803594866e-09j)
CompOverlap 4
ExpandSol milieu S= [[ 1.        +0.00000000e+00j  0.        -6.62551023e-10j
   0.0041057 +3.81651401e-09j -0.06429601-1.35750885e-09j]
 [ 0.        +6.62551023e-10j  1.        +0.00000000e+00j
   0.06429601+1.21408718e-09j  0.0041057 -3.81651401e-09j]
 [ 0.0041057 -3.81651401e-09j  0.06429601-1.21408718e-09j
   1.        +0.00000000e+00j  0.        -2.84290888e-09j]
 [-0.06429601+1.35750885e-09j  0.0041057 +3.81651401e-09j
   0.        +2.84290888e-09j  1.        +0.00000000e+00j]]
Solution syst linéaire  0  =  [-0.78372807-1.68345434e-05j]
Solution syst linéaire  1  =  [-0.65781565-3.85065395e-08j]
CompOverlap 4
ExpandSol final S= [[ 1.        +0.00000000e+00j  0.        +1.79636529e-05j
  -0.04053443-1.04033255e-06j  0.22320359+1.72631341e-05j]
 [ 0.        -1.79636529e-05j  1.        +0.00000000e+00j
  -0.22320359+1.73005876e-05j -0.04053443+1.04033255e-06j]
 [-0.04053443+1.04033255e-06j -0.22320359-1.73005876e-05j
   1.        +0.00000000e+00j  0.        +5.46899250e-08j]
 [ 0.22320359-1.72631341e-05j -0.04053443-1.04033255e-06j
   0.        -5.46899250e-08j  1.        +0.00000000e+00j]]
Fock [[-3.11736311-2.53473399e-04j  0.135721  -2.00341228e-05j]
 [ 0.13104901-8.90746170e-06j -0.50348839+1.09819859e-05j]]
Kinetic [[3.63597781-5.54727183e-04j 0.0756795 -2.11244374e-05j]
 [0.07569298-2.09642584e-05j 1.53173611+2.84598506e-09j]]
Vnuc [[-8.41030887-4.34024976e-19j  0.08657582-2.00892001e-05j]
 [ 0.08657582+2.00892001e-05j -3.26321206-6.15422376e-22j]]
Coulomb [[ 3.31393578+3.25194477e-19j -0.05309527+5.56079596e-06j]
 [-0.05309527-5.56079596e-06j  1.09124169-3.30872245e-24j]]
Exchange [[ 1.65696789+2.76873895e-20j -0.02188896+2.40553110e-06j]
 [-0.02654747+1.35982686e-05j -0.13674587-1.09791399e-05j]]
Fock [[-3.11736311-2.53473399e-04j  0.135721  -2.00341228e-05j]
 [ 0.13104901-8.90746170e-06j -0.50348839+1.09819859e-05j]]
Kinetic [[3.63597781-5.54727183e-04j 0.0756795 -2.11244374e-05j]
 [0.07569298-2.09642584e-05j 1.53173611+2.84598506e-09j]]
Vnuc [[-8.41030887-4.34024976e-19j  0.08657582-2.00892001e-05j]
 [ 0.08657582+2.00892001e-05j -3.26321206-6.15422376e-22j]]
Coulomb [[ 3.31393578+3.25194477e-19j -0.05309527+5.56079596e-06j]
 [-0.05309527-5.56079596e-06j  1.09124169-3.30872245e-24j]]
Exchange [[ 1.65696789+2.76873895e-20j -0.02188896+2.40553110e-06j]
 [-0.02654747+1.35982686e-05j -0.13674587-1.09791399e-05j]]
Orbital: 0    Norm: [0.21870783 0.39954654]    Update: [0.78312283 0.66826715]    Energy:[-3.12525418-1.15602279e-08j -0.79160869+1.26428217e-10j]
Orbital: 1    Norm: [0.21870783 0.39954654]    Update: [0.78312283 0.66826715]    Energy:[-3.12525418-1.15602279e-08j -0.79160869+1.26428217e-10j]
=============Iteration: 1
F= [[-3.11736311-2.53473399e-04j  0.135721  -2.00341228e-05j]
 [ 0.13104901-8.90746170e-06j -0.50348839+1.09819859e-05j]]
CompOverlap 4
S= [[ 1.        +0.00000000e+00j  0.        -1.79636529e-05j
  -0.04053443+1.04033255e-06j  0.22320359-1.72631341e-05j]
 [ 0.        +1.79636529e-05j  1.        +0.00000000e+00j
  -0.22320359-1.73005876e-05j -0.04053443-1.04033255e-06j]
 [-0.04053443-1.04033255e-06j -0.22320359+1.73005876e-05j
   1.        +0.00000000e+00j  0.        -5.46899250e-08j]
 [ 0.22320359+1.72631341e-05j -0.04053443+1.04033255e-06j
   0.        +5.46899250e-08j  1.        +0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe011108700>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110daa30>
Orthonormalise GS (-9.148815511596242e-10-2.209913514895149e-06j) (8.271399624254494e-12+3.5488965010433226e-05j) (-8.271399624254494e-12-3.5488965010433226e-05j)
CompOverlap 4
ExpandSol initial S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00-1.79636529e-05j
  -9.14881551e-10-2.20991351e-06j  5.42897906e-09+3.57568856e-05j]
 [ 0.00000000e+00+1.79636529e-05j  1.00000000e+00+0.00000000e+00j
   8.27139962e-12-3.54889650e-05j -9.14881551e-10+2.20991351e-06j]
 [-9.14881551e-10+2.20991351e-06j  8.27139962e-12+3.54889650e-05j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00+3.83517813e-08j]
 [ 5.42897906e-09-3.57568856e-05j -9.14881551e-10-2.20991351e-06j
   0.00000000e+00-3.83517813e-08j  1.00000000e+00+0.00000000e+00j]]
CompOverlap 4
ExpandSol pre orthonorm S= [[ 4.00845039e+00+0.00000000e+00j  5.55111512e-17+2.69842643e-05j
  -8.94165341e-02-1.52445057e-05j -5.43537790e-01-1.67016350e-04j]
 [ 5.55111512e-17-2.69842643e-05j  4.00845039e+00+0.00000000e+00j
   5.43537800e-01+1.46643830e-04j -8.94165341e-02+1.52445057e-05j]
 [-8.94165341e-02+1.52445057e-05j  5.43537800e-01-1.46643830e-04j
   5.42870066e+00+0.00000000e+00j  0.00000000e+00-3.87794752e-05j]
 [-5.43537790e-01+1.67016350e-04j -8.94165341e-02-1.52445057e-05j
   0.00000000e+00+3.87794752e-05j  5.42870066e+00+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0110da5e0>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110da5e0>
Orthonormalise GS (-0.009644743306706007+6.747276811673919e-06j) (0.05862766086502455+2.020771133434036e-05j) (-0.05862766086502455-2.020771133434036e-05j)
CompOverlap 4
ExpandSol milieu S= [[ 1.        +0.00000000e+00j  0.        +6.73184439e-06j
  -0.00964474+6.74727681e-06j -0.05862766+1.95896048e-05j]
 [ 0.        -6.73184439e-06j  1.        +0.00000000e+00j
   0.05862766-2.02077113e-05j -0.00964474-6.74727681e-06j]
 [-0.00964474-6.74727681e-06j  0.05862766+2.02077113e-05j
   1.        +0.00000000e+00j  0.        +5.92466729e-06j]
 [-0.05862766-1.95896048e-05j -0.00964474+6.74727681e-06j
   0.        -5.92466729e-06j  1.        +0.00000000e+00j]]
Solution syst linéaire  0  =  [0.06611878+0.00062074j 0.3723063 -0.00104664j]
Solution syst linéaire  1  =  [0.0216536 +1.95997274e-05j 0.43969247-2.18818496e-05j]
CompOverlap 4
ExpandSol final S= [[ 1.        +0.00000000e+00j  0.        -1.17788141e-05j
  -0.00269279-8.58629250e-07j -0.06503834+3.22917805e-05j]
 [ 0.        +1.17788141e-05j  1.        +0.00000000e+00j
   0.06503833+4.47819322e-05j -0.00269279+8.58629250e-07j]
 [-0.00269279+8.58629250e-07j  0.06503833-4.47819322e-05j
   1.        +0.00000000e+00j  0.        -8.80574937e-06j]
 [-0.06503834-3.22917805e-05j -0.00269279-8.58629250e-07j
   0.        +8.80574937e-06j  1.        +0.00000000e+00j]]
Fock [[-3.1244164 +1.97951930e-03j  0.01660659-2.76490999e-05j]
 [ 0.01958222+9.39020911e-05j -0.79646838+1.44462974e-05j]]
Kinetic [[3.61910903+4.36984475e-03j 0.35801638+2.25041638e-04j]
 [0.35804038+2.12611173e-04j 0.60665024+8.91648329e-09j]]
Vnuc [[-8.39592349+4.33342056e-18j -0.40301968+2.48583153e-04j]
 [-0.40301968-2.48583153e-04j -2.13124188+4.24575265e-20j]]
Coulomb [[3.30478849+1.75335820e-18j 0.12317185-7.79907855e-05j]
 [0.12317185+7.79907855e-05j 0.75987588+3.17637355e-21j]]
Exchange [[1.65239424-1.31374810e-18j 0.05858634-4.11219642e-05j]
 [0.06158596+6.96679051e-05j 0.03175262-1.44373809e-05j]]
Fock [[-3.1244164 +1.97951930e-03j  0.01660659-2.76490999e-05j]
 [ 0.01958222+9.39020911e-05j -0.79646838+1.44462974e-05j]]
Kinetic [[3.61910903+4.36984475e-03j 0.35801638+2.25041638e-04j]
 [0.35804038+2.12611173e-04j 0.60665024+8.91648329e-09j]]
Vnuc [[-8.39592349+4.33342056e-18j -0.40301968+2.48583153e-04j]
 [-0.40301968-2.48583153e-04j -2.13124188+4.24575265e-20j]]
Coulomb [[3.30478849+1.75335820e-18j 0.12317185-7.79907855e-05j]
 [0.12317185+7.79907855e-05j 0.75987588+3.17637355e-21j]]
Exchange [[1.65239424-1.31374810e-18j 0.05858634-4.11219642e-05j]
 [0.06158596+6.96679051e-05j 0.03175262-1.44373809e-05j]]
Orbital: 0    Norm: [1.06463191 1.01198258]    Update: [0.08471106 0.21047561]    Energy:[-3.11736311-2.52258074e-04j -0.65968082+1.55024887e-05j]
Orbital: 1    Norm: [1.06463191 1.01198258]    Update: [0.08471106 0.21047561]    Energy:[-3.11736311-2.52258074e-04j -0.65968082+1.55024887e-05j]
=============Iteration: 2
F= [[-3.1244164 +1.97951930e-03j  0.01660659-2.76490999e-05j]
 [ 0.01958222+9.39020911e-05j -0.79646838+1.44462974e-05j]]
CompOverlap 4
S= [[ 1.        +0.00000000e+00j  0.        +1.17788141e-05j
  -0.00269279+8.58629250e-07j -0.06503834-3.22917805e-05j]
 [ 0.        -1.17788141e-05j  1.        +0.00000000e+00j
   0.06503833-4.47819322e-05j -0.00269279-8.58629250e-07j]
 [-0.00269279-8.58629250e-07j  0.06503833+4.47819322e-05j
   1.        +0.00000000e+00j  0.        +8.80574937e-06j]
 [-0.06503834+3.22917805e-05j -0.00269279+8.58629250e-07j
   0.        -8.80574937e-06j  1.        +0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0110c5e80>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110daa60>
Orthonormalise GS (2.019754259837119e-09-1.812914086725065e-07j) (7.496263063197159e-09+7.72375231269468e-05j) (-7.496263063197159e-09-7.72375231269468e-05j)
CompOverlap 4
ExpandSol initial S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+1.17788141e-05j
   2.01975426e-09-1.81291409e-07j -1.00271078e-07+7.79109611e-05j]
 [ 0.00000000e+00-1.17788141e-05j  1.00000000e+00+0.00000000e+00j
   7.49626306e-09-7.72375231e-05j  2.01975426e-09+1.81291409e-07j]
 [ 2.01975426e-09+1.81291409e-07j  7.49626306e-09+7.72375231e-05j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00-8.63942675e-06j]
 [-1.00271078e-07-7.79109611e-05j  2.01975426e-09-1.81291409e-07j
   0.00000000e+00+8.63942675e-06j  1.00000000e+00+0.00000000e+00j]]
CompOverlap 4
ExpandSol pre orthonorm S= [[ 3.99908541+0.00000000e+00j  0.        +1.34577495e-06j
  -0.00690977+2.37497841e-04j -0.50766467-7.69969319e-04j]
 [ 0.        -1.34577495e-06j  3.99908541+0.00000000e+00j
   0.50766426+9.46075608e-04j -0.00690977-2.37497841e-04j]
 [-0.00690977-2.37497841e-04j  0.50766426-9.46075608e-04j
   4.57390375+0.00000000e+00j  0.        -1.16046182e-04j]
 [-0.50766467+7.69969319e-04j -0.00690977+2.37497841e-04j
   0.        +1.16046182e-04j  4.57390375+0.00000000e+00j]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0110c3340>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110da880>
Orthonormalise GS (-0.0008120612847047261-7.814007223680231e-05j) (0.05965972988950281+6.978479797322013e-05j) (-0.05965972988950281-6.978479797322013e-05j)
CompOverlap 4
ExpandSol milieu S= [[ 1.00000000e+00+0.00000000e+00j  0.00000000e+00+3.36520682e-07j
  -8.12061285e-04-7.81400722e-05j -5.96596991e-02+8.02884911e-05j]
 [ 0.00000000e+00-3.36520682e-07j  1.00000000e+00+0.00000000e+00j
   5.96597299e-02-6.97847980e-05j -8.12061285e-04+7.81400722e-05j]
 [-8.12061285e-04+7.81400722e-05j  5.96597299e-02+6.97847980e-05j
   1.00000000e+00+0.00000000e+00j  0.00000000e+00+2.77647500e-05j]
 [-5.96596991e-02-8.02884911e-05j -8.12061285e-04-7.81400722e-05j
   0.00000000e+00-2.77647500e-05j  1.00000000e+00+0.00000000e+00j]]
Solution syst linéaire  0  =  [ 8.64333728e-03+0.01428923j -1.08939952e+01+0.73304626j
 -5.34753635e+00+0.34502651j]
Solution syst linéaire  1  =  [ 0.05478482+1.49289139e-06j -1.80773394+7.12870559e-03j
 -1.39627923+3.96966153e-03j]
CompOverlap 4
ExpandSol final S= [[ 1.        +0.j          0.        -0.05828445j  0.02331196+0.00176734j
  -0.06187565-0.00017295j]
 [ 0.        +0.05828445j  1.        +0.j          0.06189128+0.00052447j
   0.02331196-0.00176734j]
 [ 0.02331196-0.00176734j  0.06189128-0.00052447j  1.        +0.j
   0.        -0.0012663j ]
 [-0.06187565+0.00017295j  0.02331196+0.00176734j  0.        +0.0012663j
   1.        +0.j        ]]
Fock [[-3.12481316-0.0187465j  -0.07145331+0.02970266j]
 [-0.06952692+0.006575j   -0.78470898-0.0001289j ]]
Kinetic [[3.60401626-4.27586823e-02j 0.07611847+2.06693996e-02j]
 [0.07613964+1.81501673e-02j 0.63862563-2.03610327e-09j]]
Vnuc [[-8.38367433+1.11022302e-16j -0.19745526+1.26614379e-02j]
 [-0.19745526-1.26614379e-02j -2.17070588+7.11507676e-19j]]
Coulomb [[3.2915868 +5.55111512e-17j 0.09952364-1.07397749e-03j]
 [0.09952364+1.07397749e-03j 0.77997351-5.07160977e-19j]]
Exchange [[1.6457934 -3.98986399e-17j 0.04770362-4.64428963e-04j]
 [0.04966133-2.31399552e-02j 0.03260224+1.28900489e-04j]]
Fock [[-3.12481316-0.0187465j  -0.07145331+0.02970266j]
 [-0.06952692+0.006575j   -0.78470898-0.0001289j ]]
Kinetic [[3.60401626-4.27586823e-02j 0.07611847+2.06693996e-02j]
 [0.07613964+1.81501673e-02j 0.63862563-2.03610327e-09j]]
Vnuc [[-8.38367433+1.11022302e-16j -0.19745526+1.26614379e-02j]
 [-0.19745526-1.26614379e-02j -2.17070588+7.11507676e-19j]]
Coulomb [[3.2915868 +5.55111512e-17j 0.09952364-1.07397749e-03j]
 [0.09952364+1.07397749e-03j 0.77997351-5.07160977e-19j]]
Exchange [[1.6457934 -3.98986399e-17j 0.04770362-4.64428963e-04j]
 [0.04966133-2.31399552e-02j 0.03260224+1.28900489e-04j]]
Orbital: 0    Norm: [1.1210345 1.0902142]    Update: [0.44959946 0.2058486 ]    Energy:[-3.1244164 +1.97946468e-03j -0.75439723+2.66848199e-05j]
Orbital: 1    Norm: [1.1210345 1.0902142]    Update: [0.44959946 0.2058486 ]    Energy:[-3.1244164 +1.97946468e-03j -0.75439723+2.66848199e-05j]
=============Iteration: 3
F= [[-3.12481316-0.0187465j  -0.07145331+0.02970266j]
 [-0.06952692+0.006575j   -0.78470898-0.0001289j ]]
CompOverlap 4
S= [[ 1.        +0.j          0.        +0.05828445j  0.02331196-0.00176734j
  -0.06187565+0.00017295j]
 [ 0.        -0.05828445j  1.        +0.j          0.06189128-0.00052447j
   0.02331196+0.00176734j]
 [ 0.02331196+0.00176734j  0.06189128+0.00052447j  1.        +0.j
   0.        +0.0012663j ]
 [-0.06187565-0.00017295j  0.02331196-0.00176734j  0.        -0.0012663j
   1.        +0.j        ]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0110c5f10>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110dafa0>
Orthonormalise GS (6.871719972968612e-05+0.004239677970036829j) (-1.5659829877157994e-05+0.0003522942033368263j) (1.5659829877157994e-05-0.0003522942033368263j)
CompOverlap 4
ExpandSol initial S= [[ 1.00000000e+00+0.j          0.00000000e+00+0.05828445j
   6.87171997e-05+0.00423968j -2.06657725e-04+0.00078838j]
 [ 0.00000000e+00-0.05828445j  1.00000000e+00+0.j
  -1.56598299e-05-0.00035229j  6.87171997e-05-0.00423968j]
 [ 6.87171997e-05-0.00423968j -1.56598299e-05+0.00035229j
   1.00000000e+00+0.j          0.00000000e+00-0.00134646j]
 [-2.06657725e-04-0.00078838j  6.87171997e-05+0.00423968j
   0.00000000e+00+0.00134646j  1.00000000e+00+0.j        ]]
CompOverlap 4
ExpandSol pre orthonorm S= [[ 3.99831029e+00+0.j          2.22044605e-16-0.09526222j
  -1.44523375e-01-0.02990464j -5.91358982e-01-0.0037277j ]
 [ 2.22044605e-16+0.09526222j  3.99831029e+00+0.j
   5.91104605e-01-0.00166854j -1.44523375e-01+0.02990464j]
 [-1.44523375e-01+0.02990464j  5.91104605e-01+0.00166854j
   4.92143048e+00+0.j          0.00000000e+00+0.00807798j]
 [-5.91358982e-01+0.0037277j  -1.44523375e-01-0.02990464j
   0.00000000e+00-0.00807798j  4.92143048e+00+0.j        ]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0110c5f10>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110daca0>
Orthonormalise GS (-0.016454695652584707-0.0013660754578833285j) (0.06714419173277979+0.001035765315959395j) (-0.06714419173277979-0.001035765315959395j)
CompOverlap 4
ExpandSol milieu S= [[ 1.        +0.j          0.        -0.02382562j -0.0164547 -0.00136608j
  -0.06703289+0.00052411j]
 [ 0.        +0.02382562j  1.        +0.j          0.06714419-0.00103577j
  -0.0164547 +0.00136608j]
 [-0.0164547 +0.00136608j  0.06714419+0.00103577j  1.        +0.j
   0.        -0.00066545j]
 [-0.06703289-0.00052411j -0.0164547 -0.00136608j  0.        +0.00066545j
   1.        +0.j        ]]
Solution syst linéaire  0  =  [-0.13926065+1.45829151j -0.23887443+0.73216686j  1.35593185-2.44036813j]
Solution syst linéaire  1  =  [ 0.72557925-0.10023841j  0.11720788-0.05242671j -1.64350787+0.22664488j]
CompOverlap 4
ExpandSol final S= [[ 1.        +0.j          0.        +0.08040901j -0.06265614-0.00766347j
  -0.04924655-0.00263309j]
 [ 0.        -0.08040901j  1.        +0.j          0.04979866+0.00562934j
  -0.06265614+0.00766347j]
 [-0.06265614+0.00766347j  0.04979866-0.00562934j  1.        +0.j
   0.        +0.00095749j]
 [-0.04924655+0.00263309j -0.06265614-0.00766347j  0.        -0.00095749j
   1.        +0.j        ]]
Fock [[-3.11764011-0.07809621j  0.20545512-0.04833163j]
 [ 0.21085073+0.00157325j -0.80031772+0.00281424j]]
Kinetic [[3.60136692-1.71233623e-01j 0.37814012-6.00625844e-02j]
 [0.3771751 -1.29877912e-03j 0.56368393-1.44953177e-07j]]
Vnuc [[-8.39390128-2.84494650e-16j -0.16125603+3.18980019e-02j]
 [-0.16125603-3.18980019e-02j -2.11556599-4.77048956e-18j]]
Coulomb [[ 3.3036943 -3.46944695e-18j -0.02027812-1.66681085e-02j]
 [-0.02027812+1.66681085e-02j  0.77947584-1.73472348e-18j]]
Exchange [[ 1.65184715+1.21430643e-17j -0.01533576-8.47780996e-03j]
 [-0.00981417+3.18029570e-02j  0.02791151-2.81438379e-03j]]
Fock [[-3.11764011-0.07809621j  0.20545512-0.04833163j]
 [ 0.21085073+0.00157325j -0.80031772+0.00281424j]]
Kinetic [[3.60136692-1.71233623e-01j 0.37814012-6.00625844e-02j]
 [0.3771751 -1.29877912e-03j 0.56368393-1.44953177e-07j]]
Vnuc [[-8.39390128-2.84494650e-16j -0.16125603+3.18980019e-02j]
 [-0.16125603-3.18980019e-02j -2.11556599-4.77048956e-18j]]
Coulomb [[ 3.3036943 -3.46944695e-18j -0.02027812-1.66681085e-02j]
 [-0.02027812+1.66681085e-02j  0.77947584-1.73472348e-18j]]
Exchange [[ 1.65184715+1.21430643e-17j -0.01533576-8.47780996e-03j]
 [-0.00981417+3.18029570e-02j  0.02791151-2.81438379e-03j]]
Orbital: 0    Norm: [0.99971653 0.99132728]    Update: [0.40052142 0.15815419]    Energy:[-3.12481316-0.01652159j -0.7381471 -0.00011843j]
Orbital: 1    Norm: [0.99971653 0.99132728]    Update: [0.40052142 0.15815419]    Energy:[-3.12481316-0.01652159j -0.7381471 -0.00011843j]
=============Iteration: 4
F= [[-3.11764011-0.07809621j  0.20545512-0.04833163j]
 [ 0.21085073+0.00157325j -0.80031772+0.00281424j]]
CompOverlap 4
S= [[ 1.        +0.j          0.        -0.08040901j -0.06265614+0.00766347j
  -0.04924655+0.00263309j]
 [ 0.        +0.08040901j  1.        +0.j          0.04979866-0.00562934j
  -0.06265614-0.00766347j]
 [-0.06265614-0.00766347j  0.04979866+0.00562934j  1.        +0.j
   0.        -0.00095749j]
 [-0.04924655-0.00263309j -0.06265614+0.00766347j  0.        +0.00095749j
   1.        +0.j        ]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0128ae040>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110da250>
Orthonormalise GS (-0.0009536486354003712-0.01578278537422972j) (-0.0005538209660944085+0.0030055424614537085j) (0.0005538209660944085-0.0030055424614537085j)
CompOverlap 4
ExpandSol initial S= [[ 1.00000000e+00+0.j          0.00000000e+00-0.08040901j
  -9.53648635e-04-0.01578279j -5.70826981e-04+0.00269557j]
 [ 0.00000000e+00+0.08040901j  1.00000000e+00+0.j
  -5.53820966e-04-0.00300554j -9.53648635e-04+0.01578279j]
 [-9.53648635e-04+0.01578279j -5.53820966e-04+0.00300554j
   1.00000000e+00+0.j          0.00000000e+00+0.00196722j]
 [-5.70826981e-04-0.00269557j -9.53648635e-04-0.01578279j
   0.00000000e+00-0.00196722j  1.00000000e+00+0.j        ]]
CompOverlap 4
ExpandSol pre orthonorm S= [[ 4.00162907e+00+0.j          5.55111512e-17+0.13526653j
   1.41156345e-01+0.02222365j -5.94005022e-01+0.003099j  ]
 [ 5.55111512e-17-0.13526653j  4.00162907e+00+0.j
   5.93167735e-01-0.01734246j  1.41156345e-01-0.02222365j]
 [ 1.41156345e-01-0.02222365j  5.93167735e-01+0.01734246j
   4.62004530e+00+0.j          0.00000000e+00-0.01175017j]
 [-5.94005022e-01-0.003099j    1.41156345e-01+0.02222365j
   0.00000000e+00+0.01175017j  4.62004530e+00+0.j        ]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe011108ac0>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110da250>
Orthonormalise GS (0.016670578535009903-0.00012005866165091481j) (0.06971340108590014+0.0013054110398602694j) (-0.06971340108590014-0.0013054110398602694j)
CompOverlap 4
ExpandSol milieu S= [[ 1.        +0.00000000e+00j  0.        +3.38028658e-02j
   0.01667058-1.20058662e-04j -0.06953047-6.14824194e-05j]
 [ 0.        -3.38028658e-02j  1.        +0.00000000e+00j
   0.0697134 -1.30541104e-03j  0.01667058+1.20058662e-04j]
 [ 0.01667058+1.20058662e-04j  0.0697134 +1.30541104e-03j
   1.        +0.00000000e+00j  0.        +1.84579698e-03j]
 [-0.06953047+6.14824194e-05j  0.01667058-1.20058662e-04j
   0.        -1.84579698e-03j  1.        +0.00000000e+00j]]
Solution syst linéaire  0  =  [-0.13907241+0.0692109j   0.85529584-0.48431377j  0.19380311+0.33517009j]
Solution syst linéaire  1  =  [ 0.34389559+0.14361095j -2.71136114-0.54091315j  1.19609413+0.0038564j ]
CompOverlap 4
ExpandSol final S= [[ 1.        +0.j          0.        +0.27387988j -0.01067794-0.00153276j
  -0.06297342+0.00173444j]
 [ 0.        -0.27387988j  1.        +0.j          0.06133319+0.00353288j
  -0.01067794+0.00153276j]
 [-0.01067794+0.00153276j  0.06133319-0.00353288j  1.        +0.j
   0.        +0.13001016j]
 [-0.06297342-0.00173444j -0.01067794-0.00153276j  0.        -0.13001016j
   1.        +0.j        ]]
Fock [[-3.06096423-0.04016986j  0.04010592-0.14252272j]
 [ 0.04977847-0.05978667j -0.78454857+0.00165579j]]
Kinetic [[3.4735738 -1.25630652e-01j 0.48560197-1.63151432e-03j]
 [0.46739475-2.01434545e-01j 0.61997668+8.85966750e-07j]]
Vnuc [[-8.39171357-4.71844785e-16j -0.48265263-2.20673635e-01j]
 [-0.48265263+2.20673635e-01j -2.18808451-2.02962647e-16j]]
Coulomb [[3.30248318-4.16333634e-17j 0.11243528+5.16262502e-02j]
 [0.11243528-5.16262502e-02j 0.81901539-6.72205347e-17j]]
Exchange [[1.65124159+7.63278329e-17j 0.04948899+2.44890453e-02j]
 [0.05707148+1.10135556e-01j 0.03545613-1.65490610e-03j]]
Fock [[-3.06096423-0.04016986j  0.04010592-0.14252272j]
 [ 0.04977847-0.05978667j -0.78454857+0.00165579j]]
Kinetic [[3.4735738 -1.25630652e-01j 0.48560197-1.63151432e-03j]
 [0.46739475-2.01434545e-01j 0.61997668+8.85966750e-07j]]
Vnuc [[-8.39171357-4.71844785e-16j -0.48265263-2.20673635e-01j]
 [-0.48265263+2.20673635e-01j -2.18808451-2.02962647e-16j]]
Coulomb [[3.30248318-4.16333634e-17j 0.11243528+5.16262502e-02j]
 [0.11243528-5.16262502e-02j 0.81901539-6.72205347e-17j]]
Exchange [[1.65124159+7.63278329e-17j 0.04948899+2.44890453e-02j]
 [0.05707148+1.10135556e-01j 0.03545613-1.65490610e-03j]]
Orbital: 0    Norm: [0.99660183 1.15089044]    Update: [0.11692138 0.4961442 ]    Energy:[-3.11764011-0.06976473j -0.756002  +0.000498j  ]
Orbital: 1    Norm: [0.99660183 1.15089044]    Update: [0.11692138 0.4961442 ]    Energy:[-3.11764011-0.06976473j -0.756002  +0.000498j  ]
=============Iteration: 5
F= [[-3.06096423-0.04016986j  0.04010592-0.14252272j]
 [ 0.04977847-0.05978667j -0.78454857+0.00165579j]]
CompOverlap 4
S= [[ 1.        +0.j          0.        -0.27387988j -0.01067794+0.00153276j
  -0.06297342-0.00173444j]
 [ 0.        +0.27387988j  1.        +0.j          0.06133319-0.00353288j
  -0.01067794-0.00153276j]
 [-0.01067794-0.00153276j  0.06133319+0.00353288j  1.        +0.j
   0.        -0.13001016j]
 [-0.06297342+0.00173444j -0.01067794+0.00153276j  0.        +0.13001016j
   1.        +0.j        ]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0111081c0>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe011108580>
Orthonormalise GS (-0.0011932913749520923-0.0034000079223216097j) (0.0016435567332675476+0.005278010151579514j) (-0.0016435567332675476-0.005278010151579514j)
CompOverlap 4
ExpandSol initial S= [[ 1.        +0.j          0.        -0.27387988j -0.00119329-0.00340001j
  -0.00268812+0.00071381j]
 [ 0.        +0.27387988j  1.        +0.j          0.00164356-0.00527801j
  -0.00119329+0.00340001j]
 [-0.00119329+0.00340001j  0.00164356+0.00527801j  1.        +0.j
   0.        +0.12978166j]
 [-0.00268812-0.00071381j -0.00119329-0.00340001j  0.        -0.12978166j
   1.        +0.j        ]]
CompOverlap 4
ExpandSol pre orthonorm S= [[ 4.04005058+0.j          0.        +0.45663707j  0.16423978+0.12030674j
  -0.70142686-0.00071391j]
 [ 0.        -0.45663707j  4.04005058+0.j          0.68315646-0.02023198j
   0.16423978-0.12030674j]
 [ 0.16423978-0.12030674j  0.68315646+0.02023198j  4.94457479+0.j
   0.        -0.60929603j]
 [-0.70142686+0.00071391j  0.16423978+0.12030674j  0.        +0.60929603j
   4.94457479+0.j        ]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe011108e50>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110dadc0>
Orthonormalise GS (0.016095496568680652+0.025451091224759956j) (0.08163786746384859+0.002434065331196212j) (-0.08163786746384859-0.002434065331196212j)
CompOverlap 4
ExpandSol milieu S= [[ 1.        +0.j          0.        +0.11302756j  0.0160955 +0.02545109j
  -0.07675258+0.00076683j]
 [ 0.        -0.11302756j  1.        +0.j          0.08163787-0.00243407j
   0.0160955 -0.02545109j]
 [ 0.0160955 -0.02545109j  0.08163787+0.00243407j  1.        +0.j
   0.        +0.11791549j]
 [-0.07675258-0.00076683j  0.0160955 +0.02545109j  0.        -0.11791549j
   1.        +0.j        ]]
Solution syst linéaire  0  =  [-0.00681959+0.68277853j  0.78123717-0.58655613j  0.05167019+0.06712591j]
Solution syst linéaire  1  =  [ 1.83716536+0.47832297j -0.40867012-0.35740921j -2.65047553-1.41418062j]
CompOverlap 4
ExpandSol final S= [[ 1.        +0.j          0.        -0.46742392j  0.01997352+0.0363173j
  -0.07389734+0.00811068j]
 [ 0.        +0.46742392j  1.        +0.j          0.08087411-0.00686816j
   0.01997352-0.0363173j ]
 [ 0.01997352-0.0363173j   0.08087411+0.00686816j  1.        +0.j
   0.        +0.55864704j]
 [-0.07389734-0.00811068j  0.01997352+0.0363173j   0.        -0.55864704j
   1.        +0.j        ]]
Fock [[-2.93113817-0.11225201j -0.05764093+0.33091189j]
 [-0.06858331+0.01520925j -0.79016559-0.00370653j]]
Kinetic [[ 3.17541113-2.13680478e-01j -0.02468045-3.29067571e-02j]
 [-0.02363465+4.05627302e-01j  0.61039301+3.46239277e-07j]]
Vnuc [[-8.38421676-1.11022302e-15j -0.0501777 +3.07540503e-01j]
 [-0.0501777 -3.07540503e-01j -2.1832727 -2.22044605e-16j]]
Coulomb [[3.296885  +2.77555756e-16j 0.03738441-3.88076839e-02j]
 [0.03738441+3.88076839e-02j 0.82344001-4.16333634e-17j]]
Exchange [[1.6484425 -1.38777878e-17j 0.01960814-1.91857107e-02j]
 [0.02121299-1.94017411e-01j 0.04072591+3.70687722e-03j]]
Fock [[-2.93113817-0.11225201j -0.05764093+0.33091189j]
 [-0.06858331+0.01520925j -0.79016559-0.00370653j]]
Kinetic [[ 3.17541113-2.13680478e-01j -0.02468045-3.29067571e-02j]
 [-0.02363465+4.05627302e-01j  0.61039301+3.46239277e-07j]]
Vnuc [[-8.38421676-1.11022302e-15j -0.0501777 +3.07540503e-01j]
 [-0.0501777 -3.07540503e-01j -2.1832727 -2.22044605e-16j]]
Coulomb [[3.296885  +2.77555756e-16j 0.03738441-3.88076839e-02j]
 [0.03738441+3.88076839e-02j 0.82344001-4.16333634e-17j]]
Exchange [[1.6484425 -1.38777878e-17j 0.01960814-1.91857107e-02j]
 [0.02121299-1.94017411e-01j 0.04072591+3.70687722e-03j]]
Orbital: 0    Norm: [1.02006947 1.04559633]    Update: [0.4640799  0.49653382]    Energy:[-3.06096423-0.03616572j -0.7376608 +0.00527787j]
Orbital: 1    Norm: [1.02006947 1.04559633]    Update: [0.4640799  0.49653382]    Energy:[-3.06096423-0.03616572j -0.7376608 +0.00527787j]
=============Iteration: 6
F= [[-2.93113817-0.11225201j -0.05764093+0.33091189j]
 [-0.06858331+0.01520925j -0.79016559-0.00370653j]]
CompOverlap 4
S= [[ 1.        +0.j          0.        +0.46742392j  0.01997352-0.0363173j
  -0.07389734-0.00811068j]
 [ 0.        -0.46742392j  1.        +0.j          0.08087411+0.00686816j
   0.01997352+0.0363173j ]
 [ 0.01997352+0.0363173j   0.08087411-0.00686816j  1.        +0.j
   0.        -0.55864704j]
 [-0.07389734+0.00811068j  0.01997352-0.0363173j   0.        +0.55864704j
   1.        +0.j        ]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe0110c5ac0>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110c58b0>
Orthonormalise GS (0.007538986973476779+0.07261766397220921j) (-0.0069834777208093324+0.0012437118899293227j) (0.0069834777208093324-0.0012437118899293227j)
CompOverlap 4
ExpandSol initial S= [[ 1.        +0.j          0.        +0.46742392j  0.00753899+0.07261766j
  -0.02699259+0.00509752j]
 [ 0.        -0.46742392j  1.        +0.j         -0.00698348-0.00124371j
   0.00753899-0.07261766j]
 [ 0.00753899-0.07261766j -0.00698348+0.00124371j  1.        +0.j
   0.        +0.55811762j]
 [-0.02699259-0.00509752j  0.00753899+0.07261766j  0.        -0.55811762j
   1.        +0.j        ]]
CompOverlap 4
ExpandSol pre orthonorm S= [[ 4.13127249+0.j          0.        -0.84230534j -0.04860816-0.12492344j
  -0.87124288-0.02603152j]
 [ 0.        +0.84230534j  4.13127249+0.j          0.925942  +0.00443742j
  -0.04860816+0.12492344j]
 [-0.04860816+0.12492344j  0.925942  -0.00443742j  5.17232861+0.j
   0.        -2.53279301j]
 [-0.87124288+0.02603152j -0.04860816-0.12492344j  0.        +2.53279301j
   5.17232861+0.j        ]]
====================WARNING: GRAM SCHMIDT ORTHOGONALISATION =========================
CompOverlap 4
<class 'numpy.complex128'> <class 'spinor.spinor'> <class 'float'> <class 'spinor.spinor'>
test1 <class 'spinor.spinor'> <spinor.spinor object at 0x7fe011108a90>
test2 <class 'spinor.spinor'>
test3 <spinor.spinor object at 0x7fe0110daa30>
Orthonormalise GS (-0.003206776444266958-0.002577094922725249j) (0.09114688718772643+0.00522918363654273j) (-0.09114688718772643-0.00522918363654273j)
CompOverlap 4
ExpandSol milieu S= [[ 1.        +0.j          0.        -0.20388521j -0.00320678-0.00257709j
  -0.10282864+0.00427937j]
 [ 0.        +0.20388521j  1.        +0.j          0.09114689-0.00522918j
  -0.00320678+0.00257709j]
 [-0.00320678+0.00257709j  0.09114689+0.00522918j  1.        +0.j
   0.        +0.5069742j ]
 [-0.10282864-0.00427937j -0.00320678-0.00257709j  0.        -0.5069742j
   1.        +0.j        ]]
Solution syst linéaire  0  =  [ 0.34265832-0.62433925j -0.21921542+0.41165831j  0.61987219+0.0339217j ]
Solution syst linéaire  1  =  [-0.25060824-0.32347402j  0.03475089+0.07260906j  0.73369754-0.44002147j]
CompOverlap 4
ExpandSol final S= [[ 1.        +0.j          0.        -0.3560941j   0.03473546-0.00636947j
  -0.09106198+0.00246364j]
 [ 0.        +0.3560941j   1.        +0.j          0.08433182-0.00313309j
   0.03473546+0.00636947j]
 [ 0.03473546+0.00636947j  0.08433182+0.00313309j  1.        +0.j
   0.        +0.22773736j]
 [-0.09106198-0.00246364j  0.03473546-0.00636947j  0.        -0.22773736j
   1.        +0.j        ]]